<html>
<head>
<title> Biology Syllabus </title>
<meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="../notescss.css">
<script src="https://code.jquery.com/jquery-1.10.2.js"></script>


</head>
<body> 
	<form>
		<input type="button" class="back" value=" " onclick="history.back()">
	   </form> 
  <div id="nav-placeholder">

  </div>


    <script>
    $(function(){
      $("#nav-placeholder").load("../nav.html");
    });
    </script>
    <!--end of Navigation bar-->


      <!--

  <div class="dropdown">
    <button class="dropbtn">Methods
      <i class="fa fa-caret-down"></i>
    </button>
    <div class="dropdown-content">
      <a href="c:\\Users/emman\Documents\Coding\methods/methods_syllabus.html">Syllabus</a>
      <a href="c:\\Users/emman\Documents\Coding\methods/methods_notes.html">Notes</a>
  </div>
  </div>
  <div class="dropdown">
    <button class="dropbtn">Specialist
      <i class="fa fa-caret-down"></i>
    </button>
    <div class="dropdown-content">
      <a href="c:\\Users/emman\Documents\Coding\specialist/specialist_syllabus.html">Syllabus</a>
      <a href="c:\\Users/emman\Documents\Coding\specialist/specialist_notes.html">Notes</a>
  </div>
  </div> -->

  <a href="javascript:void(0);" style="font-size:15px;" class="icon" onclick="myFunction()">&#9776;</a>
</div>



<script>
function myFunction() {
  var x = document.getElementById("myTopnav");
  if (x.className === "topnav") {
    x.className += " responsive";
  } else {
    x.className = "topnav";
  }
}
</script>

<script>
var prevScrollpos = window.pageYOffset;
window.onscroll = function() {
var currentScrollPos = window.pageYOffset;
  if (prevScrollpos > currentScrollPos) {
    document.getElementById("myTopnav").style.top = "0";
  } else {
    document.getElementById("myTopnav").style.top = "-50px";
  }
  prevScrollpos = currentScrollPos;
}
</script>

<div class="everything" >
  <br><BR><BR><BR>
<h1 class="extp"> Syllabus Objectives and Link to Notes for Biology</h1>

	<h2 class="extp"> UNIT 3 </h2>

	<div>
	<br>

<h2 class="topic"> Topic 1: Describing Biodiversity </h2>



<table class="center">
  <tr>
    <th>Subject Matter </th>
	<th> Guidance </th>
        <tr>
		<td>
			<div class="sub"> Biodiversity
			</div>
							<ul class="list">
				<li> <a href="biology_notes.html#3.1.1A" id="3.1.1A"> recognise that biodiversity includes the diversity of species and ecosystems
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.1B" > determine diversity of species using measures such as species richness,
evenness (relative species abundance), percentage cover, percentage
frequency and Simpson’s diversity index
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.1C" > use species diversity indices, species interactions (predation, competition,
symbiosis, disease) and abiotic factors (climate, substrate, size/depth of area)
to compare ecosystems across spatial and temporal scales
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.1D" > explain how environmental factors limit the distribution and abundance of
species in an ecosystem
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.1E" > Mandatory practical: Determine species diversity of a group of organisms
based on a given index
					</a>
				</li>
				</ul>
		</td>
		<td>
				<ul class="list">
				<li>
					<a href="biology_notes.html"> Use local context throughout the unit to develop the content objectives
					</a>
				</li>
				<li>
					<a href="biology_notes.html">  Diversity indices and measurements should be supported through fieldwork
and based on classification. Measures of biodiversity, i.e. species richness (S)
and Simpson’s diversity index (D) should be used where applicable.
					</a>
				</li>
				<li>
					<a href="biology_notes.html"> Formula: The formula used to quantify biodiversity of a habitat is Simpson’s
diversity index (SDI), shown as:
					</a>
				</li>
				<li>
					<a href="biology_notes.html">  Manipulative skill: Use appropriate technology, such as data loggers,
chemical tests, turbidity tubes and other equipment to measure factors.
					</a>
				</li>
				</ul>
		</td>
		</tr>

		<tr>
		<td>
			<div class="sub"> Classification Process
			</div>
			<ul class="list">
				<li> <a href="biology_notes.html#3.1.2A" id="3.1.2A"> recognise that biological classification can be hierarchical and based on
								different levels of similarity of physical features, methods of reproduction and
								molecular sequences
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2B" > describe the classification systems for
						<ul>
							<li> similarity of physical features (the Linnaean system) </li>
							<li>methods of reproduction (asexual, sexual — K and r selection)</li>
							<li>molecular sequences (molecular phylogeny — also called cladistics)</li>
						</ul>
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2C" > define the term clade
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2D" > recall that common assumptions of cladistics include a common ancestry,
bifurcation and physical change
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2E" > interpret cladograms to infer the evolutionary relatedness between groups of
organisms
					</a>
				</li>
				<li> <a href="biology_notes.html#3.1.2F"> analyse data from molecular sequences to infer species evolutionary
relatedness
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2G" > recognise the need for multiple definitions of species
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2H" > identify one example of an interspecific hybrid that does not produce fertile
offspring (e.g. mule, Equus mulus)
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2I" > explain the classification of organisms according to the following species
interactions: predation, competition, symbiosis and disease
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2J" > understand that ecosystems are composed of varied habitats (microhabitat to
ecoregion)
					</a>
				</li>
				<li> <a href="biology_notes.html#3.1.2K"> interpret data to classify and name an ecosystem
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2L" > explain how the process of classifying ecosystems is an important step
towards effective ecosystem management (consider old-growth forests,
productive soils and coral reefs)
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2M" > describe the process of stratified sampling in terms of
						<ul>
							<li> purpose (estimating population, density, distribution, environmental
gradients and profiles, zonation, stratification) </li>
							<li> site selection </li>
							<li> choice of ecological surveying technique (quadrats, transects) </li>
							<li> minimising bias (size and number of samples, random-number generators,
counting criteria, calibrating equipment and noting associated precision) </li>
							<li> methods of data presentation and analysis. </li>
						</ul>
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2N" > Mandatory practical: Use the process of stratified sampling to collect and
analyse primary biotic and abiotic field data to classify an ecosystem
					</a>
				</li>
			</ul>
		</td>
		<td>
				<ul class="list">
				<li>
					<a href="biology_notes.html#3.1.2A"> Students should understand that the concept of classification is directly
related to the purpose for which the data will be used.
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2B"> Students should recognise that the Linnean system does not rely solely on
physical features for classification
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2C"> Classification should be supported by the analysis of field data
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2C"> Students should recognise that conserved sequences (e.g. mitochondrial
DNA) are assumed to accumulate mutations at a constant rate over time and,
therefore, provide a method for dating divergence.
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2E"> Identification of applications of molecular phylogeny (DNA barcoding and genetic testing) should be linked to understanding of subject matter in Unit 4.
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2F"> Refer to the glossary for a definition of clade
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2G"> Students should be familiar with the limitations of different definitions of
species, e.g. biological species concept and phylogenetic species concept.
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2H"> Classification of ecosystems could be based on the Holdridge life zone
classification scheme, Specht’s classification system, ANAE classification
system or EUNIS habitat classification system.
					</a>
				</li>
				</ul>
		</td>
		</tr>
	</table>
	<br>



<h2 class="topic"> Topic 2: Ecosystem Dynamics </h2>

	<table class="center">
  <tr>
    <th>Subject Matter </th>
	<th> Guidance </th>
        <tr>
		<td>
			<div class="sub"> Functioning ecosystems
			</div>
			<ul class="list">
				<li> <a href="biology_notes.html#3.2.1A" id="3.2.1A"> sequence and explain the transfer and transformation of solar energy into
biomass as it flows through biotic components of an ecosystem, including
						<ul>
							<li> converting light to chemical energy  </li>
							<li> producing biomass and interacting with components of the carbon cycle </li>
						</ul>
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.1B" > analyse and calculate energy transfer (food chains, webs and pyramids) and
transformations within ecosystems, including
						<ul>
							<li> loss of energy through radiation, reflection and absorption  </li>
							<li>efficiencies of energy transfer from one trophic level to another </li>
							<li> biomass</li>
						</ul>
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.1C" > construct and analyse simple energy-flow diagrams illustrating the movement
of energy through ecosystems, including the productivity (gross and net) of
the various trophic levels
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.1D" > describe the transfer and transformation of matter as it cycles through
ecosystems (water, carbon and nitrogen)
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.1E" > define ecological niche in terms of habitat, feeding relationships and
interactions with other species
					</a>
				</li>
				<li> <a href="biology_notes.html#3.2.1F"> understand the competitive exclusion principle
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.1G" > analyse data to identify species (including microorganisms) or populations
occupying an ecological niche
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.1H" > define keystone species and understand the critical role they play in
maintaining the structure of a community
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.1I" > analyse data (from an Australian ecosystem) to identify a keystone species
and predict the outcomes of removing the species from an ecosystem.
					</a>
				</li>
				</ul>
		</td>
		<td>
				<ul class="list">
				<li>
					<a href="biology_notes.html#3.2.2J">  Energy transfers through ecosystems should be demonstrated using food
chains, webs and pyramids
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.1K"> A detailed description and understanding of the multiple biochemical steps in
photosynthesis and respiration (from Unit 1) is not required. Gross inputs and
outputs are required.
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.1L"> Interactions between the biotic and abiotic components of the ecosystem
should be represented through food webs, biomass pyramids, the water cycle
and biogeochemical cycles — carbon and nitrogen.
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2M" > Fieldwork should be used to develop scientific skills and collect data, as well
as to develop student understanding of concepts, especially abiotic–biotic
relationships and biotic–biotic relationships.
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.1.2N"> Refer to the glossary for a definition of ecological niche and keystone species
					</a>
				</li>
				</ul>
		</td>
		</tr>

		<tr>
		<td>


	<div class="sub"> Population Ecology
			</div>
			<ul class="list">
				<li> <a href="biology_notes.html#3.2.2A" id="3.2.2A"> define the term carrying capacity
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.2B" > explain why the carrying capacity of a population is determined by limiting
factors (biotic and abiotic)
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.2C" >  calculate population growth rate and change (using birth, death, immigration
and emigration data)
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.2D" > use the Lincoln Index to estimate population size from secondary or primary
data
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.2E" > analyse population growth data to determine the mode (exponential growth
J-curve, logistic growth S-curve) of population growth
					</a>
				</li>
				<li> <a href="biology_notes.html#3.2.2F"> discuss the effect of changes within population-limiting factors on the carrying
capacity of the ecosystem
					</a>
				</li>
				</ul>

		</td>
		<td>
				<ul class="list">
				<li>
					<a href="biology_notes.html">  Refer to the glossary for a definition of carrying capacity.
					</a>
				</li>
				<li>
					<a href="biology_notes.html"> Limiting factors of population growth should include
						<ul>
							<li>biotic factors — competition for resources, predation and disease</li>
							<li>abiotic factors — space, availability of nutrients, pollution, natural
disasters, extreme climatic events (drought, cyclones, global temperature change).</li>
						</ul>
					</a>
				</li>
				<li>
					<a href="biology_notes.html"> Formula: The formula for estimation of population size by the capture–
recapture measure is the Lincoln index, shown as
					</a>
				</li>
				</ul>
		</td>
		</tr>
		<tr>
		<td>

		<div class="sub"> Changing ecosystems
			</div>
			<ul class="list">
				<li> <a href="biology_notes.html#3.2.3A" id="3.2.3A"> explain the concept of ecological succession (refer to pioneer and climax
communities and seres)
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.3B" >  differentiate between the two main modes of succession: primary and
secondary
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.3C" >  identify the features of pioneer species (ability to fixate nitrogen, tolerance to
extreme conditions, rapid germination of seeds, ability to photosynthesise)
that make them effective colonisers
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.3D" > analyse data from the fossil record to observe past ecosystems and changes
in biotic and abiotic components
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.3E" > analyse ecological data to predict temporal and spatial successional changes
					</a>
				</li>
				<li> <a href="biology_notes.html#3.2.3F"> predict the impact of human activity on the reduction of biodiversity and on the
magnitude, duration and speed of ecosystem change.
					</a>
				</li>
				<li>
					<a href="biology_notes.html#3.2.3G" > Mandatory practical: Select and appraise an ecological surveying technique to analyse species diversity between two spatially variant ecosystems of the
same classification (e.g. a disturbed and undisturbed dry sclerophyll forest).
					</a>
				</li>
				</ul>

		</td>
		<td>
				<ul class="list">
				<li>
					<a href="biology_notes.html">  Predictions of succession could be based on r-selected species versus
K-selected species, biodiversity, biomass, or changes in biotic and abiotic
interactions.
					</a>
				</li>
				<li>
					<a href="biology_notes.html"> Human activities could include overexploitation, habitat destruction,
monocultures or pollution.
					</a>
				</li>
				</ul>
		</td>
		</tr>

	</table>

		</div>


		<!-- yadad
		adf
		fad

		dataad
		addressfad
		fadfa
		dfa
		datad
		addressd
		datada
		dataadda
		datadad
		ga
		ffhf
		hjdfg
		gf
		jg
		hgsfjf
		dg
		h1gh
		gh
		fertile
		-->


	<h2 class="extp" id="unit4"> UNIT 4: Heridity and Continuity of Life</h2>
	<div>
	<br>

<h2 class="topic"> Topic 1: DNA, gense and the continuity of life </h2>



<table class="center">
  <tr>
    <th>Subject Matter </th>
	<th> Guidance </th>
        <tr>
		<td>
			<div class="sub"> DNA Structure and Replication
			</div>
							<ul class="list">
				<li> <a href="biology_notes.html#4.1.1A" id="4.1.1A"> understand that deoxyribonucleic acid (DNA) is a double-stranded molecule
that occurs bound to proteins (histones) in chromosomes in the nucleus, and
as unbound circular DNA in the cytosol of prokaryotes, and in the
mitochondria and chloroplasts of eukaryotic cells.
					</a>
				</li>
				<li>
					<a href="biology_notes.html#4.1.1B" >recall the structure of DNA, including:<br>
					<uL>
 <li> nucleotide composition</li>
 <li>complementary base pairing</li>
 <li>weak, base-specific hydrogen bonds between DNA strands</li>
 </ul>
					</a>
				</li>
				<li>
					<a href="biology_notes.html#4.1.1C" > explain the role of helicase (in terms of unwinding the double helix and
separation of the strands) and DNA polymerase (in terms of formation of the
new complementary strands) in the process of DNA replication. Reference
should be made to the direction of replication.
					</a>
				</li>
				</ul>
		</td>
		<td>
				<ul class="list">
				<li>
					<a href="biology_notes.html">  Identification and use of chemical formulas are not required for recalling the
components in DNA structure. Students should be able to use a schematic
model identifying nucleotides (nitrogenous base + phosphate + sugar) and the
associated hydrogen bonds
					</a>
				</li>
				<li>
					<a href="biology_notes.html"> Specific numbers of hydrogen bonds and reference to purines and
pyrimidines are not required for the description of hydrogen bonding in DNA.
					</a>
				</li>
				<li>
					<a href="biology_notes.html"> Reference to DNA polymerase I and II is not required in the explanation of
DNA replication.
					</a>
				</li>
				</ul>
		</td>
		</tr>
		<tr>
		<td>
			<div class="sub"> Cellular Replication and Variation
			</div>
							<ul class="list">
				<li> <a href="biology_notes.html#4.1.2A" id="4.1.2A"> compare and contrast the process of meiosis I and II
­ recognise the role of homologous chromosomes
­ describe the processes of crossing over and recombination and
demonstrate how they contribute to genetic variation
­ compare and contrast the process of spermatogenesis and oogenesis
(with reference to haploid and diploid cells).
					</a>
				</li>
				<li>
					<a href="biology_notes.html#4.1.2B" >demonstrate how the process of independent assortment and random
fertilisation alter the variations in the genotype of offspring.
					</a>
				</li>
				</ul>
		</td>
		<td>
		</td>
		</tr>
		<tr>
		<td>
			<div class="sub"> Gene Expression
			</div>
							<ul class="list">
				<li> <a href="biology_notes.html#4.1.3A" id="4.1.3A"> define the terms genome and gene
				</a> </li>
				<li> <a href="biology_notes.html#4.1.3B"> understand that genes include ‘coding’ (exons) and ‘noncoding’ DNA (which
includes a variety of transcribed proteins: functional RNA (i.e. tRNA),
centromeres, telomeres and introns. Recognise that many functions of
‘noncoding’ DNA are yet to be determined)
				</a> </li>
				<li> <a href="biology_notes.html#4.1.3C"> explain the process of protein synthesis in terms of
­ transcription of a gene into messenger RNA in the nucleus
­ translation of mRNA into an amino acid sequence at the ribosome (refer to
transfer RNA, codons and anticodons)
				</a> </li>
				<li> <a href="biology_notes.html#4.1.3D"> recognise that the purpose of gene expression is to synthesise a functional
gene product (protein or functional RNA); that the process can be regulated
and is used by all known life
				</a> </li>
				<li> <a href="biology_notes.html#4.1.3E"> Identify that there are factors that regulate the phenotypic expression of genes
<ul> <li> During transcription and translation (proteins that bind to specific DNA
sequences)</li> <li>
Through the products of other genes</li> <li>
Via environmental exposure (consider the twin methodology in epigenetic
studies)</li> </uL>
				</a> </li>
				<li> <a href="biology_notes.html#4.1.3F"> recognise that differential gene expression, controlled by transcription factors,
regulates cell differentiation for tissue formation and morphology
				</a> </li>
				<li> <a href="biology_notes.html#4.1.3G"> recall an example of a transcription factor gene that regulates morphology
(HOX transcription factor family) and cell differentiation (sex-determining
region Y).
				</a> </li>
				</ul>
		</td>
		<td>
			<ul class="list">
				<li> <a href="biology_notes.html"> Refer to glossary for definitions of genome and gene
				</a> </li>
				<li> <a href="biology_notes.html"> The term junk DNA is misleading and should not be used in reference to
‘noncoding’ DNA.
				</a> </li>
				<li> <a href="biology_notes.html"> When identifying transcription factors in the regulation of gene expression,
reference to operators, promoters, regulators, enhancers, silencers,
insulators, TATA boxes, polyadenylation and DNA methylation is not required.
				</a> </li>
				<li> <a href="biology_notes.html"> Students should recognise gene expression in the context of an example.
They are not required to explain or describe the mechanisms of this process
				</a> </li>
				</ul>
		</td>
		</TR>
		<tr>
		<td>
			<div class="sub"> Mutations
			</div>
							<ul class="list">
				<li> <a href="biology_notes.html#4.1.4A" id="4.1.4A">Identify how mutations in genes and chromosomes can result from errors in:
<ul> <li> DNA replication (point and frameshift mutation)
</li> <li> Cell division (non-disjunction)
</li> <li> Damage by mutagens (physical, including UV radiation, ionising radiation
</li> <li> And heat and chemical)</li> </ul>

				</a> </li>
				<li> <a href="biology_notes.html#4.1.4B"> explain how non-disjunction leads to aneuploidy
				</a> </li>
				<li> <a href="biology_notes.html#4.1.4C"> use a human karyotype to identify ploidy changes and predict a genetic
disorder from given data
				</a> </li>
				<li> <a href="biology_notes.html#4.1.4D"> describe how inherited mutations can alter the variations in the genotype of
offspring.
				</a> </li>
				</ul>
		</td>
		<td>
			<ul class="list">
				<li> <a href="biology_notes.html"> Students are not required to identify the effects of mutations (i.e. silent,
missense, nonsense).
				</a> </li>
				<li> <a href="biology_notes.html"> Recall of specific chemical mutagens is not required. Rather, an
understanding should be developed that a large number of chemical
mutagens are carcinogenic and interact directly with DNA.
				</a> </li>
				<li> <a href="biology_notes.html"> Examples of aneuploidy could include trisomy 21.

				</a> </li>
				</ul>
		</td>
		</TR>

		<tr>
		<td>
			<div class="sub"> Inheritance
			</div>
							<ul class="list">
				<li> <a href="biology_notes.html#4.1.5A" id="4.1.5A"> predict frequencies of genotypes and phenotypes using data from probability
models (including frequency histograms and Punnett squares) and by taking
into consideration patterns of inheritance for the following types of alleles:
autosomal dominant, sex linked and multiple
				</a></li>
				<li> <a href="biology_notes.html#4.1.5B"> define polygenic inheritance and predict frequencies of genotypes and
phenotypes for using three of the possible alleles.
				</a></li>
				</ul>
		</td>
		<td>
				<ul class="list">
				<li>
					<a href="biology_notes.html"> Refer to the glossary for a definition of polygenic inheritance.
					</a> </li>
					<li>
					<a href="biology_notes.html"> Multiple allele inheritance refers to situations where there are more than two
alleles considered, i.e. incomplete and co-dominance situations.
					</a> </li>
					<li>
					<a href="biology_notes.html"> Examples for patterns of inheritance could include haemophilia (sex linked)
and ABO blood types (multiple), grain colour in wheat (polygenic).
					</a> </li>
					</ul>
		</td>
</tr>


		<tr>
		<td>
			<div class="sub"> Biotechnology
			</div>
							<ul class="list">

				<li> <a href="biology_notes.html#4.1.6A" id="4.1.6A"> describe the process of making recombinant DNA
<ul> <li> isolation of DNA, cutting of DNA (restriction enzymes)
</li> <li>insertion of DNA fragment (plasmid vector)
</li> <li> joining of DNA (DNA ligase)
</li> <li> amplification of recombinant DNA (bacterial transformation)</li> </ul>
				</a></li>
				<li> <a href="biology_notes.html#4.1.6B"> recognise the applications of DNA sequencing to map species’ genomes and
DNA profiling to identify unique genetic information
				</a></li>
				<li> <a href="biology_notes.html#4.1.6C"> explain the purpose of polymerase chain reaction (PCR) and gel
electrophoresis
				</a></li>
				<li> <a href="biology_notes.html#4.1.6D"> appraise data from an outcome of a current genetic biotechnology technique
to determine its success rate.

				</a></li>
				</ul>
				</td>

				<td>
				<ul class="list">
				<li>
					<a href="biology_notes.html"> Examples for species genome mapping could include the Human Genome
Project. The BLAST database could be used to develop understanding of
bioinformatics.
					</a> </li>
					<li>
					<a href="biology_notes.html"> Examples of current biotechnology techniques could include gel
electrophoresis, PCR or CRISPR-based technologies.
					</a> </li>
					<li>
					<a href="biology_notes.html"> Data for appraisal could be from DNA banding, frequency of DNA fragments,
effectiveness of restriction enzymes, location of a gene or gene expression.
This could be supported with a practical.
					</a> </li>

					</ul>
					</td>
				</tr>
				</table>







<h2 class="topic"> Topic 2: Continuuity of life on Earth </h2>



<table class="center">
  <tr>
    <th>Subject Matter </th>
	<th> Guidance </th>
        <tr>
		<td>
			<div class="sub"> Evolution
			</div>
							<ul class="list">
				<li> <a href="biology_notes.html#4.2.1A" id="4.2.1A"> define the terms evolution, microevolution and macroevolution
					</a>
				</li>
				<li>
					<a href="biology_notes.html#4.2.1B" >
determine episodes of evolutionary radiation and mass extinctions from an
evolutionary timescale of life on Earth (approximately 3.5 billion years)
					</a>
				</li>
				<li>
					<a href="biology_notes.html#4.2.1C" > interpret data (i.e. degree of DNA similarity) to reveal phylogenetic
relationships with an understanding that comparative genomics involves the
comparison of genomic features to provide evidence for the theory of
evolution.
					</a>
				</li>
				</ul>
		</td>
		<td>
				<ul class="list">
				<li>
					<a href="biology_notes.html">  Refer to the glossary for definitions of evolution, microevolution and
macroevolution

					</a>
				</li>
				<li>
					<a href="biology_notes.html"> Evolutionary radiation refers to an increase in taxonomic diversity or
morphological disparity.
					</a>
				</li>

				</ul>
		</td>
		</tr>
		<tr>
		<td>
			<div class="sub"> Natural selection and microevolution

			</div>
							<ul class="list">
				<li> <a href="biology_notes.html#4.2.2A" id="4.2.2A"> recognise natural selection occurs when the pressures of environmental
selection confer a selective advantage on a specific phenotype to enhance its
survival (viability) and reproduction (fecundity)
					</a>
				</li>
				<li>
					<a href="biology_notes.html#4.2.2B" >identify that the selection of allele frequency in a gene pool can be positive or
negative
					</a>
				</li>

				<li>
					<a href="biology_notes.html#4.2.2C" >interpret data and describe the three main types of phenotypic selection:
stabilising, directional and disruptive
				<li>
					<a href="biology_notes.html#4.2.2D" > explain microevolutionary change through the main processes of mutation,
gene flow and genetic drift.
					</a>
				</li>
				<li>
					<a href="biology_notes.html#4.2.2E" > Mandatory practical: Analyse genotypic changes for a selective pressure in
a gene pool (modelling can be based on laboratory work or computer
simulation).
					</a>
				</li>

				</ul>
		</td>
		<td>
		<ul class="list">
				<li> <a href="biology_notes.html"> Examples of natural selection could include beak size in the Galapagos
finches, antibiotic resistance or insecticide resistance
				</a> </li>
				<li> <a href="biology_notes.html"> Recognise that mutation is the ultimate source of genetic variation, as it
introduces new alleles to a population (syllabus link to Unit 4, Topic 1).

				</a> </li>
				</ul>
		</td>
		</tr>
		<tr>
		<td>
			<div class="sub"> Speciation and macroevolution
			</div>
							<ul class="list">
				<li> <a href="biology_notes.html#4.2.3A" id="4.2.3A"> recall that speciation and macroevolutionary changes result from an
accumulation of microevolutionary changes over time
				</a> </li>
				<li> <a href="biology_notes.html#4.2.3B"> identify that diversification between species can follow one of four patterns:
divergent, convergent, parallel and coevolution
				</a> </li>
				<li> <a href="biology_notes.html#4.2.3C"> describe the modes of speciation: allopatric, sympatric, parapatric
				</a> </li>
			<li> <a href="biology_notes.html#4.2.3D">understand that the different mechanisms of isolation — geographic (including
environmental disasters, habitat fragmentation), reproductive, spatial, and
temporal — influence gene flow
				</a> </li>
				<li> <a href="biology_notes.html#4.2.3E"> explain how populations with reduced genetic diversity (i.e. those affected by
population bottlenecks) face an increased risk of extinction
				</a> </li>
				<li> <a href="biology_notes.html#4.2.3F"> interpret gene flow and allele frequency data from different populations in
order to determine speciation.
				</a> </li>

				</ul>
		</td>
		<td>
			<ul class="list">
				<li> <a href="biology_notes.html"> Habitat fragmentation should be referred to in terms of natural and human
causes.
				</a> </li>
				<li> <a href="biology_notes.html"> Populations with reduced genetic diversity should be linked to subject matter
in Unit 3.

				</a> </li>
				<li> <a href="biology_notes.html">Students should be able to determine the modes of speciation from
interpretation of data.
				</a> </li>
				</ul>
		</td>
		</TR>
		</div>

				</table>


       		</div>

          <div id="div-placeholder">

          </div>

              <script>
              $(function(){
                $("#div-placeholder").load("../footer.html");
              });
              </script>

				</body>
				</html>
