<html>
<head> 

<meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="../notescss.css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

<title>Biology Content </title>
<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
</head>
<body>

	<form>
		<input type="button" class="back" value=" " onclick="history.back()">
	   </form> 

  <div id="nav-placeholder">

  </div>

      <script>
      $(function(){
        $("#nav-placeholder").load("../nav.html");
      });
      </script>







<div>
  <a href="javascript:void(0);" style="font-size:15px;" class="icon" onclick="myFunction()">&#9776;</a>
</div>

    <script>

    function myFunction() {
      var x = document.getElementById("myTopnav");
      if (x.className === "topnav") {
        x.className += " responsive";
      } else {
        x.className = "topnav";
      }
    }
    // Function to change webpage background color
        function changeTopNavBg(color){
    	    document.getElementById("myTopnav").style.background = color;

    	}

       /* // Function to change heading background color
       function changeTopNavHoover(color){
            document.getElementByClass("dropdown").getAttribute("hover").style.background = color;
        }*/

    	function changeStruBg(color){
            document.getElementById("structure").style.background = color;
        }



    	function changeCogverb(color) {
        var elements = document.getElementsByClassName("cogverb");
    	for(var i = 0; i < elements.length; i++){
    		elements[i].style.background = color;
    		}
    }

    function changeBox(color) {
        var elements = document.getElementsByClassName("box");
    	for(var i = 0; i < elements.length; i++){
    		elements[i].style.background = color;
    		}
    }

    function changeShade(color) {
        var elements = document.getElementsByClassName("dropbtn");
    	for(var i = 0; i < elements.length; i++){
    		elements[i].style.background = color;
    		}
    }




    	/*
    	function changTopNavHoover(color){
    	document.getElementById("dropbtn").addEventListener("mouseover", mouseOver);
       document.getElementById("dropbtn").addEventListener("mouseout", mouseOut);
    	}

       function mouseOver() {
       document.getElementById("dropbtn").style.color = "red";
       }

       function mouseOut() {
        document.getElementById("dropbtn").style.color = "black";
        }*/
    </script>





    <script>
    function myFunction() {
      var x = document.getElementById("myTopnav");
      if (x.className === "topnav") {
        x.className += " responsive";
      } else {
        x.className = "topnav";
      }
    }
    var prevScrollpos = window.pageYOffset;
    window.onscroll = function() {
    var currentScrollPos = window.pageYOffset;
      if (prevScrollpos > currentScrollPos) {
        document.getElementById("myTopnav").style.top = "0";
      } else {
        document.getElementById("myTopnav").style.top = "-50px";
      }
      prevScrollpos = currentScrollPos;
    }
    </script>
    <script>

    var articles = document.getElementsByTagName('')
    for (var i = 0; i < articles.length; i ++) {
        articles[i].parentNode.insertBefore(document.createElement('hr'), articles[i].nextSibling)
    }
    </script>
  </div>












<div class="structure" id="structure">
<div class="everythink">


    	<div id="mySidebar" class="sidebar">
    		<a href="javascript:void(0)" class="closebtn" onclick="closeNav()">×</a>
        <a href="snr_biology_19_syll.pdf" target="_blank">Biology Syllabus</a>
    		<a href=#unit3 >Unit 3</a>
    		<a href=#unit4>Unit 4</a>
    		<a href="#"></a>
    	</div>



    	<script>
    	function openNav() {
    		document.getElementById("mySidebar").style.width = "150px";
    		document.getElementById("main").style.marginLeft = "300px";
    		document.getElementById("main").style.display = "none";
    	}

    	function closeNav() {
    		document.getElementById("mySidebar").style.width = "0";
    		document.getElementById("main").style.marginLeft= "0";
    			document.getElementById("main").style.display = "block";
    	}
    	</script>

      <div id="main">
      <button id="opensesame" class="openbtn" onclick="openNav()">&rsaquo;</button>
     </div>




<div id="unit3" class="topic">
  <br><br>
	<h1> UNIT 3 </h1><h1>Topic 1: Describing Biodiversity </h1>
</div>
		<div class="sub-topic">
		<h2> Biodiversity </h2>
	</div>
		<div class="content">

	<strong class="cogverb" id="3.1.1A"> Recognise <span class="tooltiptext"> 3.1.1A: recognise that biodiversity includes the diversity of species and ecosystems </span></strong>	Biodiversity is the diversity of species and ecosystems.

		<ul>
			<li> <b>Species richness </b> The number of species in a region or specified area</li>
			<li> <b>Species evenness </b> the degree of equitability in the distribution of individuals among a group of species (relative species abundance). Max evenness is the same number of individuals among all species.  </li>
			<li> <b>Population </b> A group of organisms in the same species</li>
		</ul>
	<p> Biodiversity is:
		<ol>
			<li> The number and relative evenness of species within  biome, region, ecosystem, or habitat. </li>
			<li> Variety/Number of ecosystems within a region, or the variety / number of habitats within an ecosystem. </li>
		</ol>
	</p>

	<h3> &#10132; Species Diversity </h3>
		<div class="super-sub-topic">
	<strong class="cogverb" id="3.1.1B"> Determine <span class="tooltiptext"> 3.1.1B: determine diversity of species using measures such as species richness,
evenness (relative species abundance), percentage cover, percentage
frequency and Simpson’s diversity index </span></strong>  <strong class="cogverb" id="3.1.1C"> Use <span class="tooltiptext"> 3.1.1C: use species diversity indices, species interactions (predation, competition,
symbiosis, disease) and abiotic factors (climate, substrate, size/depth of area)
to compare ecosystems across spatial and temporal scales</span></strong>

		<h4> Percentage Frequency </h4>
			<p> Simplest way to collect data using frame quadrat is to make a species list for each quadrat. Each species is present of absent. This allows to calculate the percentage frequency of each species for the whole sample area. Percentage frequency is the probability that a species is found within a single quadrat. </p>


    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=default'></script>
    $$  \text{ % frequency} = {{\text{number of quadrats in which species is found} } \over {\text{total number of quadrats}}} $$


  	<h4> Percentage Cover</h4>
			<p> Percentage cover is a method of determine relative abundance based on the amount of space taken up by a species, it is usually used to measure populations of sessile (unmoving) organisms. (think what we did for bacteria plates, counting grid squares.
			</p>
			<img src="bio photos/percentage_cover.png" width=400px>

		<h4> Simpson's Biodiversity Index </h4>
			<p> Biodiversity can be quantified by statistical methods, one of which is Simpson’s Index of Diversity.
			</p>
			<img src="bio photos/simpsons_index.png" width=60%>

      <h4> Spatial and Temporal scale </h4>
      <p> <b> Spatial </b> = space </p>
      <p> <b> Temporal </b> = time </p>
		</DIV>
	<h3 id="species-interactions"> &#10132; Species Interactions</h3>
		<div class="super-sub-topic">
			<Ol>
				<li> <b>Predation </b> one organism (the predator attacks and kills another living organism (prey) to obtain food. Beneficial for predator, harmful for prey.</li>
				<li> <b>Competition</b> rivalry between individuals for a specific resource which may be intraspecific   (within a species) or interspecific (between species).  </li>
				<li> <b>Symbiosis </b> two organisms live in close association over long period time. Different types of relationships within this category: </li>
					<ol type="a">
						<li> <b>Mutualism </b> both species benefit from the obligatory interaction (e.g., bacteria living in the intestines of termites, algae in coral polyps)</li>
						<li> <b>Cooperation</b> Each species benefits from the association, but the present of one is not essential to the survival of the other (sea anemones living on the shell or crabs)</li>
						<li> <b>Commensalism </b>One organism benefits and the other is not affected. (Epiphytic ferns and orchids on rainforest trees). </li>
						<li> <b>Amensalism </b>One species inhibits the other (e.g., antibiotics produced by molds inhibit the growth of some bacteria).  </li>
						<li> <b>Parasitism </b>Parasites obtain food by living in (endoparasites e.g., tapeworm) or (ectoparasites) a most organisms. The parasite benefits while the host is harmed.  </li>
					</ol>
				<li> <b> Pathogens and Disease: </b> A pathogen is an organism that causes infectious disease. Infectious disease can be spread, directly or indirectly, form one organism to another. Agents that cause disease fall into 5 groups: virus, bacteria, fungi, and parasites</li>
			</ul>
		</div>
		<h3> &#10132; Abiotic Factors</h3>
			<div class="super-sub-topic">
			<p> In order to survive, grow and reproduce, each organism in a particular habitat must be able to survive in the physical and chemical conditions that surround it.
	These physical conditions of habitats and ecosystems vary dramatically between and within an ecosystem or habitat. Their affect on species and ecosystems as a whole an vary considerably across time and space. This variation results in the formation of different habitats and microhabitats and can affect the productivity and biodiversity of that system.
			</p>

	<h4><strong class="cogverb" id="3.1.1D"> Explain <span class="tooltiptext"> 3.1.1D: explain how environmental factors limit the distribution and abundance of
species in an ecosystem</span></strong> Limiting Factors.</h4>
		<p><b>Limiting factor:</b> A resource or environment condition which limits this growth, distribution or abundance of an organism or population within an ecosystem.
      <dl> <dt>
        Density independent factors </dt>
        <dd> abiotic factors that affect population size regardless of the population size. </dd >
          <dt> Density dependent factors </dt>
          <dd> factors that influence population regulation and has a greater impact on populations as their density increases-
            competition, predation.
          </dd>
        </dl>
    </p>

  	<p>

		<b>Abiotic factors:</b> contributes to much of variation between different ecosystems and habitats. Each habitat or microhabitat within same ecosystem may have slightly different abiotic conditions, thus restricting some parts of an ecosystem to organisms (outside of tolerance range resulting in death)
		<br>
		<br> <b>Biotic Factors: </b> Species interaction can limit the abundance and distribution of species across ecosystems. Distribution and abundance of species varies in an ecosystem (based on abiotic factors), meaning that interactions and relationships with other organisms within ecosystem vary in effect and intensity, helping to control where each species lives within a space.
		</div>
		</p>


	<div class="sub-topic">
		<h2> Classification Processes </h2>
	</div>

		<div class="content">
			<h3> &#10132; Biological Classification </h3>
				<div class="super-sub-topic">
					<h4> Hierarchical Classification </h4>
						<ul>
							<li> The method of grouping from larger groups to detailed groups is termed hierarchal. Results in groups within groups, each layer becoming increasingly more similar and includes less species
							</li>
							<li> Arranging things into groups based on observed similarities is known as classification
							</li>
						</ul>

							<strong class="cogverb" id="3.1.2A"> Recognise <span class="tooltiptext"> 3.1.2A:recognise that biological classification can be hierarchical and based on
      								different levels of similarity of physical features, methods of reproduction and
      								molecular sequences </span></strong>
                      	<dl>
								<dt> Physical Features </dt>
									<dd> Using observable structural features to separate into groups. (still plays role in modern day Linnean classification systems) </dd>
								<dt> Methods of Reproduction </dt>
									<dd> Can be based on reproductive strategies. Some organisms clone themselves (asexual) and others use genetic material from two parents to produce offspring (sexual).</dd>
								<dt> Molecular Sequences </dt>
									<dd> Based on DNA, grouping by comparing molecular sequences of different organisms. </dd>
							</dl>
						</p>
					<h4>Classification Systems </h4>
						 <strong class="cogverb" id="3.1.2B"> Describe <span class="tooltiptext"> 3.1.2B: describe the classification systems for
						<ul>
							<li> similarity of physical features (the Linnaean system) </li>
							<li>methods of reproduction (asexual, sexual — K and r selection)</li>
							<li>molecular sequences (molecular phylogeny — also called cladistics)</li>
						</ul> </span> </strong>
						<h5> Similarities of Physical Features (The Linnaean system) </h5>
							<p> <b> Taxonomy </b> is the science of naming and classifying living things based on physical and biochemical characteristics.
							</p>
							<p> Each category in the Linnaean systems represents a level of grouping from a larger general group (or taxon) to small groups with more specific characteristics.
							</p>
								<ol>
									<li> Hierarchical/rank-based structure </li>
									<li> Ranking structure was- Three kingdoms divided into <b>phyla/division, classes, orders, families, genus</b> and <b>species.</B> With an additional rank (sub-species) below species. THINK: <b> P COFG p</b>lease <b>c</b>an <b>o</b>loaf <b>f</b>ocus <b>g</b>ertrude.</li>
									<li> Uses <b>binomial nomenclature </b> to name classified organisms at the <b>Genus and species level </b>(binomial nomenclature=two words naming system) </li>
									<li> Uses morphological features of organisms to create groups according to their similarities/features of the groups, becoming increasing similar at each lower taxon. </li>
								</ol>
							<p>
								Scientific Nomenclature: <b>Genus and SPECIES </b>
							</p>
						<ol class="yeet">
						<li> 	 A group of closely related species, arising from a common ancestor, belong to the same genus (begins in capital and always placed first).
            </li> <li>
							 Specific name indicates particular species within that genus (lowercase and is meaningless without genus Infront).
							</li>
            </ol>
							</p>
						<h5> Methods of Reproduction </h5>
							<table class="table-list">
								<tr>
									<th> Asexual reproduction </th>
									<th> Sexual reproduction </th>
								</tr>
								<tr>
									<td>
										<ul>
											<li> Offspring identical to parent </li>
											<li> One parent</li>
											<li> Primary method for single-cell organisms (but also plants and fungi reproduce mostly asexually)</li>
											<li> No genetic loss or genetic variability</li>
										</ul>
									</td>
									<td>
										<ul>
											<li> Mostly multicellular organisms </li>
											<li> Organism’s form gametes (sperm or eggs) containing half the normal number of chromosomes.</li>
											<li> Combination of gametes to create completely new recombination of genes.</li>
											<li> Rearrange genome (variability)</li>
											<li> Lose in costs of meiosis and recombination </li>
										</ul>
									</td>
								</tr>
							</table>
						<p> Can also be classified as r-selected or k-selected (reproductive strategies. Classifies organisms according to number of offspring and the level of care the parents play in upbringing.
						</p>
							<table class="table-list">
								<tr>
									<th> R-Selected (RAT ATTACK &#128000;) </th>
									<th> K-Selected (&#129432;)</th>
								</tr>
								<tr>
									<td>
										<ul>
											<li> Large no. of offspring </li>
											<li>little of no parental investment</li>
											<li>low survivorship</li>
											<li>unstable environments</li>
											<li>reach maturity early</li>
											<li>reproduce once</li>
											<li>short life expectancy</li>

										</ul>
									</td>
									<td>
										<ul>
											<li>small number of offspring </li>
											<li>large amount of parental investment </li>
											<li>high survivorship (long life expectancy) </li>
											<li>large </li>
											<li>reach maturity later </li>
											<li>reproduce multiple times </li>
										</ul>
									</td>
								</tr>
							</table>
						<h5> Molecular Sequences (molecular Phylogeny, also called cladistics)</h5>
							<p> Comparison of molecular sequences of different organisms to identify relatedness and group them hieratically. Molecules used could be DNA, RNA or amin acids/polypeptides.
							</p>
							<p>Shared molecular heritage means that based (DNA and RNA) and amin acid sequences can be compared to ascertain levels or relatedness.
							</p>
							<p>The number of differences between comparable base sequences suggest more time has past since species diverged. The more similar base sequence, the more related species are.
							</p>


				</div>


			<h3> &#10132; Cladistics </h3>
				<div class="super-sub-topic">

				<h4> Cladograms/ Phylogenetic Tree </h4>
				<strong class="cogverb" id="3.1.2C"> Define
					<span class="tooltiptext"> 3.1.2C: define the term clade </span> </strong>
				<p>A clade is a group of organisms that consists of a common ancestor and all its lineal descendants</p>

		<strong class="cogverb" id="3.1.2D"> Recall <span class="tooltiptext"> 3.1.2D: recall that common assumptions of cladistics include a common ancestry,
bifurcation and physical change </span></strong> <h5> Assumptions about cladograms: </h5>
					<p>	<ul>
							<li> Any groups of organisms are related by descent from a common ancestor. </li>
							<li> There is a bifurcating pattern in cladogenesis, meaning new kinds of organisms may arise when existing populations divide into exactly two groups. </li>
							<li>Change in characteristics occurs in lineages over time. Cladistics used the term plesiomorphic for the original characteristic and apomorphic for the characteristic derived </li>
						</ul>
          </p>
						<h5> Interpreting Cladograms </h5>
            <strong class="cogverb" id="3.1.2E"> Interpret <span class="tooltiptext"> 3.1.2E: interpret cladograms to infer the evolutionary relatedness between groups of
    organisms </span></strong>

					<img src="bio photos/clad_exp.png" width=40%>
					<img src="bio photos/clad_exp2.png" width=55%>


					<p> Each branch point <b>(node)</b> represents a <a href=#4.2.3B> divergence event</a> or splitting apart into two descendant groups. At each branch point lies the most recent common ancestor of all the groups descended from that branch point.
					</p>
					<h4> Molecular Sequences </h4> <strong class="cogverb" id="3.1.2F"> Analyse <span class="tooltiptext"> 3.1.2F: analyse data from molecular sequences to infer species evolutionary
  relatedness </span></strong>
					<dl>
								<dt> Chromosomes and DNA </dt>
									<dd> The karyotype of an organism is a description of the number and arrangement of DNA into chromosomes present in the cell. Each species has a set number of chromosomes found in each cell of its body.
											Similarities and differences between species can therefore be determined by comparing their karyotypes.
											The greater the number of similar proteins, the closer the relationship between organisms. If they have similar proteins, they will have similar nucleic acids and therefore may have a common origin.
											Therefore, the analysis of molecules such as DNA or proteins can be used to determine the evolutionary relationship between organisms.
									</dd>
								<dt> Amino Acids Sequences</dt>
									<dd> Each protein is made up of a chain of amino acids (polypeptide chain)
										The greater number of differences in amino acids between two organisms, the more time has passed for changes to accumulate since common ancestor.
									</dd>
								<dt>Mitochondrial DNA</dt>
									<dd> In sexually reproducing species, all mitochondria found in cells are inherited from mother only. Using this mtDNA to determine the relationship between two groups of organisms has many advantages over the DNA found:
										<ul>
											<li> Direct genetic lines can be traced (since only maternal DNA) </li>
											<li> mtDNA does not exhibit recombination creating a garbled genetic history </li>
											<li> mtDNA has a higher rate of mutations than nuclear DNA, making it easier to compare differences between closely related individuals. When mutation rate is determined, molecular clock can be calculated to find time of divergence. </li>
											<li> Conserved sequences (e.g., mitochondrial DNA) are assumed to accumulate mutations at a constant rate over time and, therefore, provide a method for dating divergence. </li>
										</ul>
									</dd>
								</dl>
						</div>

			<h3> &#10132; The Species Concept </h3>
				<div class="super-sub-topic">

			<strong class="cogverb" id="3.1.2G"> Recognise <span class="tooltiptext"> 3.1.2G: recognise the need for multiple definitions of species
    </span></strong>
			<h4> Biological Species Concept: </h4>
			<p>
				 The definition of species used in the biological species concept: a group of similar organisms whose members are capable of exchanging genes (natural environment) to produce (viable), fertile offspring.
			</p>
				<h5> Reproductive Isolation </h5>
				<p> Different species are referred to as being reproductively isolated </p>
				<p> Reproductive isolation may result from: </p>
					<ul>
						<li>different types of behaviour.</li>
						<li>different requirements from the same environment, use of different parts of the same environment (some rainforest birds forage for insects on tree trunks whilst other species forage for insects in the canopy of the same tree) or from a physical barrier (for example a deep canyon or river) running through the environment. </li>
						<li>different arrangements of the genetic material (nucleic acid) within the cells. </li>
						<li>Chemical or physical barriers during copulation. </li>
					</ul>
			<br>
				<h5>Limitations of Biological Species Concept: </h5>
				<p> Note: in syllabus say should know these </p>
					<dl>
						<dt> Asexual Reproduction </dt>
							<dd> There are many bacteria and unicellular eukaryotes that do not form gametes such as eggs or sperm. As a result, the definition of species cannot be used for these organisms. </dd>
						<dt> Pathogenesis (apoxixis) </dt>
							<dd> The ability of an embryo to develop without fertilisation. Similar to other forms of asexual reproduction these organisms do not fit well into the biological species concept </dd>

						<dt> Ring Species&rarr; </dt>
							<dd>These connected series of neighbours (ring species) are capable of sexually reproducing with the neighbours on either side of each other. However, as the different populations get further away from each other, they become more different, until the populations at each end of the ring cannot reproduce to produce viable fertile offspring. </dd>
						<dt> Fossils </dt>
							<dd> As fossils are no longer alive, they cannot be classified on the basis of their ability to reproduce. </dd>
					</dl>
			<h4> Phylogenetic Species Concept: </h4>
				<p> The phylogenetic species concept defines a species as; an irreducible group whose members are descended from a common ancestor and who all possess a combination of certain defining, or derived, traits. Hence, this concept defines a species as a group having a shared and unique evolutionary history.</p>
			<h5> Limitations of Phylogenetic Species Concept </h5>
				<p> Because the concept defines species based on morphology, without explicitly referring to populations or reproductive boundaries, it <id="teet"> does not apply well to cases where morphologically different populations are connected by gene flow.</id></p>

			<H4> Hybridization </h4>
				<p> Closely related organisms have been known to interbreed in artificial situations (e.g., lion and tigers in zoos) or in some cases in nature, but the hybrid offspring are usually sterile. According to the syllabus document you will need to be able to</P> <strong class="cogverb" id="3.1.2H"> Identify: <span class="tooltiptext"> 3.1.2H: identify one example of an interspecific hybrid that does not produce fertile
offspring (e.g., mule, Equus mulus) </span> </strong>
A mule, Equus mulus is a cross between a male horse and female donkey (&#128014;) and is infertile </p>
			</div>

			<h3> &#10132; The Classification of Organisms </h3>
				<div class="super-sub-topic">
          <strong class="cogverb" id="3.1.2I"> Explain <span class="tooltiptext"> 3.1.2I: explain the classification of organisms according to the following species
         interactions: predation, competition, symbiosis and disease </span></strong>
			<h4> Species Interactions </h4>
				<p> This is criterion requires to explain species interactions whereas in biodiversity sub-topic you just need to use the interactions to compare ecosystems.
				</p>
					<h5> Classification by Feeding Types </h5>
						<p> Autotrophic organisms’ producer their own food; they are called <b> producers </b>.
						<br>
						Organisms that cannot make their own food and extract food from what they eat are called <b> consumers </b>.
						</p>

					<h5> Predation: </h5>
						<p>
							An interaction where one species kills and eats another, gaining energy from the interaction. Consumer/killer is the predator, food source is the prey.
							<br> Examples:
								<ul>
									<li> wedge tailed eagles and rabbits </li>
									<li> snake and mice </li>
									<li> spiders and flies </li>
								</ul>
							</p>
					<h5> Competition:</h5>
							<p>
							An interaction between organisms in which both are trying to use the same resource related to growth, reproduction or survivability is referred to as competition.
							</p>
							This can be further divided into two groups:
								<ul>
									<li> <b>Interspecific competition:</b> occurs between different species. </li>
									<li> <b>Intraspecific competition: </b>occurs between members in the same species. </li>
								</uL>
							<h6>
							Competition in different organisms:</h6>
								<table width=40%>
									<tr>
										<th>
											Animals
										</th>
										<th>
											Plants
										</th>
									</tr>
									<tr>
										<td>
											<ul>
												<li> Food </li>
												<li> Water </li>
												<li> Mates </li>
												<li> Land (territory) </li>
											</ul>
										</td>
										<td>
											<ul>
												<li> Light </li>
												<li> Water </li>
												<li> Minerals </li>
												<li> Space </li>
											</ul>
										</td>
									</table>


							</p>
					<h4> Symbiotic Relationships </h4>
			<img src="bio photos/species_interactions.png" width=30% class="handwrite left">
						<p> <b>Symbiosis: </b> two species living together. Defined as ‘a relationship in which two organisms live in close association over a long period of time
						</P>
						<p>
							<dl>
								<dt> Commensalism </dt>
									<dd>relationship between two organisms in which one organism benefits and the other is not affected.</dd>
								<dt> Parasitism </dt>
									<dd>association between organisms in which one species (the parasite) is completely dependent upon the other (the host), which is usually harmed.</dd>
								<dt> Mutualism</dt>
									<dd> necessary and positive association between two organisms of different species.</dd>
								<dt> Amensalism</dt>
									<dd> a relationship in which one species is inhibited by another, but the first is not affected or advantaged</dd>
							</dl>
						</p>

					</div>

			<h3> &#10132; Classification of Ecosystems </h3>
			<div class="super-sub-topic">

				<button> <a href="bio docs/3.1.2 j,k,l-review.docx"> Review Questions for 3.1.2 J, K, L</a> </button>
				<button> <a href="bio docs/3.1.2J,K,L-exitticket.docx"> Exit ticket for 3.1.2 J, K, L </a> </button>
				<p>Definition Check:
					<ul>
						<li> <b> Ecosystem:</b> a biological community of interacting organisms (biotic factors) and their physical environment (abiotic factors). </li>
						<li> <b> Community:</b>all the species that occupy a particular place at a given time </li>
						<li> <b> Population:</b> number of individuals of a species living in a particular place at a particular time </li>
						<li> <b> Habitat:</b> a specific location, with a particular set of biotic and abiotic conditions; where an organism normally lives. Therefore, each ecosystem can be made up of a variety of habitats.  </li>
						<li> <b> Niche:</b> a species’ role and position in its environment; a species’ interactions with the biotic and abiotic factors of its environment. </li>
						<li> <b> Microhabitat:</b> a small habitat that may be different from the surrounding larger habitat (e.g., under a rock in a stream, as opposed to on top of the rock).</li>
					</ul>
				</p>
				<p>
				<strong class="cogverb" id="3.1.2J"> Recall <span class="tooltiptext"> 3.1.2J:understand that ecosystems are composed of varied habitats (microhabitat to
ecoregion) </span></strong> Ecosystems are composed of varied habitats (a specific location, with a particular set of biotic and abiotic conditions; where an organism normally lives), from the very small microhabitats to broad ecoregions, providing opportunity for a huge variety of species and increasing biodiversity.
				</p>
				<h4> Features of Terrestrial Ecosystems </h4>
				<p> All terrestrial communities ultimately depend on the vegetation that is the producer of the ecosystem. Since plants are immobile, they are completely controlled by their environment and therefore
											strongly reflect it. For this reason, vegetation structure is an effective way of describing or categorising ecosystems
				</p>
				<p> Terrestrial communities are classified into biomes (according to the vegetation types).
				</p>
				<h4> Habitats within Habitats </h4>
				<p>
				Many biomes (and therefore ecosystems) display a series of different vertical layers on top of each other. This arrangement is called vertical stratification. This results in variation in conditions within space and an increase in the number of habitats, microhabitats and niches. Sub-communities may form within these habitats and microhabitats. This variation results in increased biodiversity.
				<br>
				</p>
				<img src="bio photos/rainforest_structure.png" width=60%>

				<h4>Ecozones and ecoregions</h4>
				<p>
				Ecozones are large areas where organisms have been evolving in relative isolation over long periods of time. They are characterized by the evolutionary history of the organisms they contain. As a result, an ecozone may contain more than one biome. Each ecozone is separated by a geographic barrier. Ecozones are further divided into ecoregions. These are geographically distinct communities based on geology, soils, climate and predominant vegetation.
				</p>
				<h5>Australian Vegetation Zones</h5>
				<img src="bio photos/zone_characteristics.png" class="left mini" width=30%><p>Another method of categorising is Australian vegetation zones. This method is based on zones of decreasing moisture from the coast to the dry interior as well as latitudinal regions. </p>

			<h4> Naming and Classifying ecosystems </h4>
				<p> As human pop grows demand for agriculture/mining increases. Identifying and classifying ecosystems is an important step
					towards effective ecosystem management. This can be done by identifying specific biotic or abiotic
					variables that are common to an ecosystem. This can be done across varying scales, in both terrestrial
					and aquatic environments and all the way from global down to microhabitat scales.
				</p>
				<p>
				<strong class="cogverb" id="3.1.2K"> Interpret <span class="tooltiptext"> 3.1.2K: interpret data to classify and name an ecosystem
				 </span></strong> Data to classify an ecosystem.
				</p>
				<h5> Holdridge life zone system- Abiotic Features</h5>
				<br>
				<img src="bio photos/triangle_hvst.png" height=300px>

				<p> Holdridge life zone system is a global system of classifying broad areas of land by climate and ecological types. This system considers:
				 <ul>
					<li> Annual precipitation </li>
					<li> Mean annual bio-temperature </li>
					<li> Potential evapotranspiration</li>
					<li> Latitudinal regions </li>
					<li> Altitudinal belts. </li>
				</ul>
				This system assumes that the soil and vegetation communities can be mapped if these climatic factors are known.
				</p>

				<h5> Specht's Classification system- Biotic Features</h5>
				<p> Another method in classifying ecosystems is to use the structural features of the plants in an area. Specht’s classification of Australian vegetation uses the foliage cover by the tallest plants in an area and the height of those plants.
				</p>
				<img src="bio photos/spechts.png">
			</div>

			<h3> &#10132; Ecosystem Management </h3>
				<div class="super-sub-topic">
				<h4> Advantages in classification of ecosystems </h4>
				<p>Consistent classification and naming of ecosystems allows scientists and conservation managers across the planet to communicate effectively, plan on a global scale, as well as consistently identify and manage similar ecosystems.
								The particular name of the ecosystem provides a general understanding of soil type, climate, topography and vegetation. These factors also determine the fauna of the area. Classification allows scientist to appropriately tailer their management practices to each ecosystem. and the needs of a range of stakeholder values. Classification allows conservation efforts to focus on specific ecosystems.
				</p>
					<ul>
						<li>Communicate accurately on a global scale. </li>
						<li>Tailer management to specific ecosystem needs and stakeholder values.  </li>
						<li>Identify and protect fragile, vulnerable and significant ecosystems (be strategic) </li>
					</ul>
				<h4>   Management of Ecosystems </h4>

				<h5> Productive Soils </h5>
				<p> <strong class="cogverb" id="3.1.2L"> Explain <span class="tooltiptext"> 3.1.2L: explain how the process of classifying ecosystems is an important step
towards effective ecosystem management (consider old-growth forests,
productive soils and coral reefs) </span></strong>
			A major reason for protection and effective management of ecosystems is the processes or resources they provide. The health of a soil is directly linked to its ability to support the growth of plants, which in turn affects the consumers that eat the plants. The quality of the soil can be improved through careful soil management.
					</p>
					<ul>
							<li>Using plants to cover the soil to keep in anchored and prevent erosion.</li>
							<li>Rotating monoculture crops with other legume-based crops (legumes containing nitrogen fixing bacteria that can increase the nitrogen levels in the soil)</li>

							<li>mulching the soil to reduce erosion and water loss.</li>
						</ul>
				<h5> Coral Reefs </h5>
				<p> Coral reefs protect coastlines from waves and storms. Coral reefs generally occur in low nutrient seas where without their ecosystem processes biodiversity and productivity would be relatively low. The high substrate complexity of corals means that they provide high densities of microhabitats and are highly productive and biodiverse. For this reason, coral reef ecosystems must be carefully managed.
				</p>
				<h5> Old Growth Forests </h5>
				<p>What is an old growth forest? Old growth forests are significant ecosystems with high biodiversity. The term old growth forest refers to the fact that a large percentage of trees are at maturity. Parts of tree decay and hollows are formed within them. A number of animal species are reliant on these types of forests because of their range of nesting hollows, variety of microclimates and greater structural complexity in comparison with forests in earlier stages of development.
				</p>
				<p> These characteristics apply to a variety of vegetation types, such as rainforests and several types of eucalypt forests. a recent global study has shown that the tall, old growth mountain ash forests of Victoria’s Central Highlands is the world’s most carbon-rich forest. The density and volume of wood produced results in the storage of massive amounts of carbon. With the world currently experiencing global warming, it is important to maintain such forests to ensure continued uptake and storage of carbon dioxide from the atmosphere
				</p>
			</div>

		<h3> &#10132; Sampling </h3>
				<div class="super-sub-topic">

        <strong class="cogverb" id="3.1.2M"> Describe <span class="tooltiptext">
          3.1.2M: describe the process of stratified sampling in terms of

          <li> purpose (estimating population, density, distribution, environmental gradients and profiles, zonation, stratification) </li>
          <li> site selection </li>
          <li> choice of ecological surveying technique (quadrats, transects) </li>
          <li> minimizing bias (size and number of samples, random-number generators, counting criteria, calibrating equipment and noting associated precision) </li>
          <li> methods of data presentation and analysis. </li>

      </span>
    </strong>
<p> PSMT BPA</p>
			<h4> Purpose of Sampling </h4>
				<p> Sampling provides info about:
					<ol>

						<dt> 1. Species population dynamics and relationships
							<dd> Ecosystem composition and structure (and therefore its classification), the relationships between its parts, its health, and the likelihood it will be able to resist change.
									There are generally too many organisms and not enough time, hence research take samples. Involves choosing a smaller area that represents the population to survey.
									Prior to each survey, researched should define purpose (independent and dependent variables)
							</dd>
						</dt>
						<dt> 2. Species population dynamics and relationships
							<dd> By measuring population size and density over time and space, ecologist can see how these populations change, measure the effect of relationships and may be able to predict how they are likely to change in the future.
								<br>To study demographics of a population, start with baseline measures:
								<ol type=a>
									<li> the number of individuals in population </li>
									<li>population density (number of individuals per area </li>
								</ol>
							</dd>
						</dt>
						<dt> 3. Population Distribution
							<dd> Ecologists want to know distribution; species dispersion patterns refer to how the individuals are distributed in a space at a give time (could be clumped, random, uniform etc.)
									Populations: environmental gradients and profiles
									An environmental gradient is a gradual change in abiotic factors through space or time. Environmental gradients can be related to factors such as altitude, temperature, depth, ocean proximity, exposure, and soil humidity. Species abundance usually change along environmental gradients in a more or less predictable way (assessing the affect of an abiotic factor along a gradient of species in a riparian habitat) </dd>
						</dt>
						<dt> 4. Communities: Zonation
							<dd> Zonation refers to the patterns that are observed in a community over a distance, based on the distinct fauna and flora found along the area (example would assessing the affect of gradient on community composition across differing zones of a rocky shore habitat. </dd>
						</dt>
						<dt> 5. Communities: Stratification
							<dd> Stratification refers to the vertical layering of a habitat, the arrangement of vegetation in layers. It classifies the layers of vegetation largely according to the different heights to which their plants grow. (Assessing vertical vegetation structure of a rainforest site by considering the vegetation cover across each stratum or layer). Used to observe the structure of communities vertically in layers. </dd>
						</dt>
						<dt> 6. Communities: Composition:
							<dd> Diversity to compare overtime or between communities. Sampling reveals which species are present in an ecosystem and helps to build a picture of community structure or identify a species of particular interest </dd>
						</dt>
					</ol>
				</p>

			<h4> Site Selection and Sampling Methods (strategies) </h4>

						<ol>
							<li> <b> Random sampling:</b>randomly choosing your samples is the best way to fairly represent the characteristics of bigger picture. Within each area, every part must have an equal chance of being chosen. (Using random number generator)</li>
							<li><b>Systematic sampling:</b> samples from a larger population are selected according to a random starting point and a fixed, periodic sampling interval (e.g., every 5m or 5th grid point). Often faster and cheaper than random, may not give true representation for population. But can also be the best is sampling of gradient. </li>
							<li><b>Stratified sampling:</b>population divided into sub-groups (strata); samples taken from stratum in proportion to its representation in total population. Random or systematic sampling is then applied. During stratified sampling, the sampling size for teach strata should be determined by identifying the fraction of the whole population (total area of all ecosystems) in each stratum. (e.g., using randomly allocated quadrates to sample diversity in a field with sheep and no sheep) </li>
						</ol>

			<h4> Surveying Techniques </h4>
				<br>

					<table>
						<tr>
							<th> Technique </th>
							<th> Description </th>
							<th> Pros </th>
							<th> Cons </th>
						</tr>
						<tr>
							<th> Point Sampling </th>
							<td>individuals are chosen using a grid reference, random number applied to map grid of transect line, and determining the proportion of the point that intercept vegetation. (Often used to collect data about vegetation distribution. </td>
							<td>Point sampling is efficient if time is limited. Good method for determining vegetation coverage and community composition. </td>
							<td>May miss organisms in low abundance or misrepresent species that are in clumped distributions. </td>
						</tr>
						<tr>
							<th>Plot/ Quadrat </th>
							<td>Within this quadrate, individual organism can be identified, counted, and measured. Often positioned systematically with a transect of grid. </td>
							<td>Can be used to determine community and population distribution, frequency of occurrence, percentage cover (plants) and biomass (if harvested). </td>
							<td>Time consuming and laborious, results depend on size, shape and number of plots used. </td>
						</tr>
						<tr>
							<th> Line Transect </th>
							<td>Sampling line placed across a community. Transects are used to determine changes in community composition (species distribution) along an environmental gradient. </td>
							<td>Give better information over larger areas and also show-up non-random distributions of some species. </td>
							<td></td>
						</tr>
					</table>
				<h5> 3 types of Line Transects </h5>
				<ol>
					<li> <b> Belt Transects </b> form of continuous quadrat sampling. Provide a more information on community composition that a line transect but can be difficult to carry out. A measured strip is located across a study area. Quadrats are used to sample the plants at regular intervals.
						<ol type=a>
							<li> Continuous &rarr; quadrats are placed adjacent to each in continuous belt</li>
							<li> Interrupted &rarr; quadrats are placed at regular intervals. </li>
						</ol>
					</li>
					<li> <b> Line intercepts </b> used to estimate relative densities of species and involve counting the number of particular species that lie on a straight line cutting across the community. </li>
					<li> <b> Strip Census </b> involves walking along an established line through an area and recording individuals observed from that line. Useful for counting vertebrates such as roadkill census, bird counts and small mammal trapping. Also useful in studies in animals that are high mobile. </li>
				</ol>


			<h4> Minimizing Bias </h4>
				<p>Bias refers to the selection for or against one particular group. It has the potential to dramatically influence the finding of an investigation. </p>
					<ol>
						<li> <b>Size and Number of Samples: </b> the sample size or number of samples should be large enough to accurately reflect the population (community) as a whole. As a general rule the larger the number (and size) of samples the closer to actual population. However, number and size of samples determined by resources and time. </li>
						<li> <b>Random sampling: </b>randomisation ensure that all organisms have equal chance o being selected. Some sample sites may be very difficult and expensive to access. Sampling sites can be determined using random generators and determined prior to surveying. </li>
						<li> <b>Counting criteria: </b> it is important to set the criteria for counting member of population or sample prior to sampling. Using quadrats, it has to be pre-set what will be counted in or out around the boundary. </li>
						<li> <b>Calibrating equipment and noting associated precision: </b> calibration is a comparison between a known measurement and the measurement in instrument. Aim is to check accuracy. Precision of equipment should be noted and considered in data. </li>
					</ol>

			<h4> Methods of Data presentation and Analysis </h4>
			<p> estimating population parameter, calculating SD and SE
with confidence intervals and correlation coefficient </p>


		</div>

		</div>
		</div>


			<div class="topic">
	<h1> Topic 2: Ecosystem Dynamics </h1>
</div>
		<div class="sub-topic">
		<h2> Functioning Eco-systems </h2>
	</div>
		<div class="content">
      <div class="super-sub-topic">
			<p> Revision Definitions </p>
				<ul>
					<li><b>Energy Transfer </b>- the movement of energy from one organism to another e.g. producer to primary consumer.</li>
					<li><b>Energy Transformation </b>- the process of changing energy from one form to another e.g., from light energy into chemical energy.</li>
					<li><b>Primary Producer </b> - An autotrophic organism (e.g., plant) producing complex organic matter, using photosynthesis</li>
					<li><b>Primary Consumer </b>- A heterotrophic organism that feeds on primary producers (e.g., animals that eat plants - herbivores).</li>
					<li><b>Secondary Consumer </b>- A heterotrophic organism that feeds on primary consumers (e.g., animals that eat other herbivorous animals - predators).</li>
					<li><b>Tertiary Consumer </b>- A heterotrophic organism that feeds on secondary consumers (e.g., animals that eat other predatory animals – apex predators).</li>
					<li><b>Decomposers </b> – a generally immobile organism, especially a soil bacterium, fungus, or invertebrate, that decomposes organic material. </li>
					<li><b>Detritivores </b>– an animal which feeds on dead organic material, especially plant detritus.</li>
					<li><b>Biomass </b> - the amount of matter that's stored in the bodies of a group of organisms. </li>
				</ul>

				<h4 style="margin-bottom: 10px;"> Ecosystem dynamics </h4>


					<strong style="width:200px;" class="cogverb" id="3.2.1A"> Sequence & Explain
					<span class="tooltiptext"> 3.2.1A: sequence and explain the transfer and transformation of solar energy into
  biomass as it flows through biotic components of an ecosystem, including
  						<ul>
  							<li> converting light to chemical energy  </li>
  							<li> producing biomass and interacting with components of the carbon cycle </li>
  						</ul></span> </strong>
					<p> Ecosystem dynamic involve two main processes: </p>
						<ol>
							<li> <b> energy flow </b> (transformation and transfer).  </li>
							<li> <b>and chemical </b>(matter) cycling <a href=#3.2.1D >(carbon and nitrogen).</a> </li>
						</ol>
					<p> Energy flows through ecosystems, while matter cycles within them. Biomass is a consequence of the build up of matter formed from the bonds made in organic molecules. </p>
					</div>

				<h3> &#10132; Energy Flow </h3>

          <div class="content">
            	<div class="super-sub-topic">
					<p> Energy flows through an ecosystem - Entering as light and exiting as heat.
					Chemicals/Matter is cycled through the system - Built into complex organic molecules (producing biomass) in photosynthesis, and broken down again into inorganic compounds in respiration </p>
				<img src="bio photos/energyflow.png" width=40%>


				<h4> Trophic Levels </h4>


					<p> <b> Trophic levels </b> describe the relative positions of producers and consumers in a food chain.
					A food chain shows a series of organisms existing in any ecosystem, through which energy is transferred.
					Each organism  derives energy from, the preceding one. </p>


				<h5> Energy Transfer </h4>
					<p> Rarely more than six link in any food chain; usually there are only three to four. This is because energy is lost to the surroundings environment in the form of heat at each level of the food chain. Only 5 to 20% of the energy contained at one level of the food chain is transferred to the next level. </p>

				<h4> Producers and Photosynthesis </h4>
				<p> Energy is changed in form (is transformed) from solar light energy into chemical energy through the process of photosynthesis by produces. About 1% of solar energy is used and transformed by producers. Productivity is a measure of the amount of energy fixed, and rate of fixation within chemical compounds at each level in an ecosystem. It can also be an indication of the amount the biomass increases over a particular length of time. In an ecosystem it is the energy limit of that system. Biomass is simply the amount of matter that's stored in the bodies of a group of organisms. </p>

				<h4> Primary Productivity </h4>
				<p> <b>Gross primary productivity </b> or GPP, is the rate at which solar energy is captured in sugar molecules during photosynthesis—energy captured per unit area per unit time, for that ecosystem. Plants typically capture and convert about 1.3% to 1.6% of the solar energy that reaches Earth's surface and use about a quarter of the captured energy for metabolism and maintenance.
				The other 98% is lost through loss of energy through radiation of heat, reflection of light back into the atmosphere and then space (e.g., ice sheets), and absorption by inorganic surfaces (e.g., substrate etc.).
				</p>

					<h5> Gross and Net Primary Production </h5>
						<p> Total primary production in an ecosystem is known as that ecosystem’s gross primary production (GPP). This is the total amount of energy taken up by the produces in a system </p>
						<P> Some of this energy will be:
						<ul>
							<li> loss of heat (during respiration) </li>
							<li> Decompose after death </li>
							<li> Removed as waster products (fecal matter) </li>
						</ul>
						</p>
						<p> Net primary production (NPP) is equal to GPP minus the energy used by the primary producers for respiration and lost to decomposers.
						Only NPP is available to consumers. This is the amount of energy stored in the bonds of the organic molecules made by the plant.
						</p>

					<h5> Energy Transfer </h5>
					<img src="bio photos/fig4.1.7.png" class="left">	<p> Energy transfer between trophic levels is usually less than 20% efficient. </p>
					<p>The <b> net secondary production </b>of an ecosystem, is the amount of chemical energy in consumers’ food that is converted to their own new biomass during a given period of time. </p>


					<h4> Consumers</h4>
						<p> Consumers use food produced by other organisms as their energy source </p>

					<h4> Decomposers and Detritivores</h4>
					<p> All organisms eventually die and fall to the environmental substrate of any ecosystem. </p>
					<p>Since only a finite (specific) amount of matter is available in the biosphere, it is recycled.
					</p>



				<h4> Food Webs </h4>
					<img src="bio photos/foodweb.png" class="left mini" width=30%>	<p> A <b> Food web </b> is a graphical representation of the many food chains that are interlinked to show the feeding relationships between organisms in an ecosystem. The arrows always point in the direction of energy movement.
					</p>
					<p> NOTE: Arrows point from food source to eater </p>
          <br><BR><BR><BR><BR><BR><BR>

        </div>
				<h3> &#10132;  Energy Flow diagrams </h3>
				<div class="super-sub-topic">
				<p> On average, only about 10% of the energy stored as biomass in one trophic level—e.g., primary producers—gets stored as biomass in the next trophic level—e.g., primary consumers. </p>
				<img src="bio photos/netprimary.png" class="left" width=30%>
				<h4> Graphical Representation of Energy Flow</h4>
				<ul>
					<li> At each level, energy is lost to the ecosystem through the movement and heat (a waste product of respiration). </li>
					<li> A similar loss in biomass and energy through sweat and feces, as well as when organisms ide and are broken down be decomposer </li>
					<li> Consequently each trophic level is smaller than the level </li>
				</ul>
				<h5> Types of ecological pyramids </h5>
					<ul>
						<li> <b> Pyramid of biomass </b> based on the total dry mass of the organism at each level </li>
						<li> <b> Pyramid of energy </b> showing the productivity of the different trophic level </li>
					</ul>

					<h4> Net productivity </h4>
					<p>  <strong style="width:25%" class="cogverb" id="3.2.1B"> Analyse and Calculate <span class="tooltiptext"> 3.2.1B: explain how the process of classifying ecosystems is an important step
towards effective ecosystem management (consider old-growth forests,
productive soils and coral reefs) </span></strong>
					<br>
					<b>Net Productivity primary consumer) = GP -(Respiration + Fecal)</b>
					</p>
					<h4> Transfer or Trophic Efficiency </h4>
					<p> The difference between the net of one trophic level and the net of the next is its transfer or trophic efficiency. This is given as a percentage transfer using the following formula.
						<br> <b>Percentage transfer = <sup> NP trophic level</sup> &frasl;<sub> NP previous trophic level x 100%</sub></b>
						<br> Using a Rainforest ecosystem </br>
            <strong class="cogverb" id="3.2.1C"> Construct and Analyse <span class="tooltiptext"> 3.2.1C: construct and analyse simple energy-flow diagrams illustrating the movement
  of energy through ecosystems, including the productivity (gross and net) of
  the various trophic levels </span></strong>
						<img src="bio photos/percentagetransfer.png" width=70%>
						</p>
						<p> The stuff on the bottom (unutilized and waste energy and meter) is lost to detritus (consumed by decomposers and detritivores). Respiration is energy/ heat loss in the atmosphere. Light is lost through reflection  </P>


				</div>
				<h3> &#10132; Cycles in ecosystems </h3>
				<div class="super-sub-topic">
					<p> Chemical elements circulate in characteristic pats, from the environment to organisms and back to the environment. There are known as biogeochemical cycles. The pathway for a particular element between living and non-living components of an ecosystem is usually referred to as a nutrient cycle. Two basic components, the reservoir pool and cycling pools. Respiration
					</p>
					Biochemical Cycles are important because:
					<ul>
						<li> They help retain necessary nutrients is usable forms </li>
						<li> They help maintain a steady state (homeostasis) in ecosystems </li>
					</ul>
					<strong class="cogverb" id="3.2.1D"> Describe <span class="tooltiptext"> 3.2.1D:describe the transfer and transformation of matter as it cycles through
ecosystems (water, carbon and nitrogen) </span></strong>
						<h4> Water Cycle </h4>
						<p> All living things need water. It is a requirement for all cells, a solvent in which many metabolic reactions take place, a reactant in a large number of chemical reactions and a habitat for marine life. </p>
						<p> The basic cycle is: <b> Evaporation/transpiration, condensation, precipitation</b>
						</p><img src="bio photos/watercycle.png">
						<h4> Carbon Cycle </h4>
							<p> Photosynthesis incorporates carbon from atmospheric carbon dioxide into complex organic molecules and oxygen is released. These compounds are broken down during cellular respiration, to release carbon dioxide and water back into the atmosphere. Over geological time, carbon is locked in a reservoir pool as coal and oil and in the wood of trees. </p>
							<p> <b> Photosynthesis and respiration cycle carbon and oxygen through the environment.
							</b></p>

							<h4> Nitrogen Cycle </h4>
              <img src="bio photos/Nitrogen-cycle-11.png" class="right" width="40%">
								<ul>
									<li> Atmospheric nitrogen (N2) makes up nearly 78%-80% of air. </li>
									<li>Organisms can not use it in that form.</li>
									<li>Lightning and bacteria convert nitrogen into usable forms.</li>
								</ul>
                <dl> <dt> assimilation </dt>
                  <dd> process by which plants and animals incorporate the NO3- and ammonia formed through nitrogen fixation and nitrification
                  </dd>
                  <dt> Denitrification </dt>
                  <dd> the process that converts nitrate to nitrogen gas
                  </dd>
                  <dt> Nitrogen fixation (ammonification) </dt>
                  <dd> converts nitrogen in the atmosphere into forms that plants can absorb through their root systems</dd>
                  <dt> Nitrification </dt>
                  <dd> the process that converts ammonia to nitrite and then to nitrate
                  </dd>
                </ul>
							<p> Nitrogen fixation-convert atmospheric nitrogen (N2) into ammonium (NH4+) which can be used to make organic compounds like amino acids.
						</p> 	<img src="bio photos/nitrogencycle.png" style="margin-left: -40px;" width=70% >

				</div>
				<h3> &#10132;Ecological Niche </h3>
				<div class="super-sub-topic">
					<strong class="cogverb" id="3.2.1E"> Define <span class="tooltiptext"> 3.2.1E: define ecological niche in terms of habitat, feeding relationships and
interactions with other species </span></strong>
						<p> The ecological niche describes the functional position and role of an organism in its environment, species’ interactions with the biotic and abiotic factors of its environment
							<ul>
								<li> The habitat in which the organism lives (abiotic factors)</li>
								<li> The resources it obtains from the habitat </li>
								<li> The interactions it has with other organisms </li>
							</ul>
						<h4> Determining Niche</h4>
						<p> A niche is determined largely by the tolerance limitations of an organism, or a limiting factor. Limiting factor- any biotic or abiotic factor that restricts the existence of organisms in a specific environment.  </p>

						<h4> LAW OF TOLERANCE </h4>
							<p> the Law of tolerance states that for each abiotic factor, an organism has a range of tolerance within which it can survive</p>
							<img src="bio photos/tolerance.png" width=50% class="float-right">

							<h4> 1. Niche: Habitat (abiotic factors)</h4>
								<p> An organism’s habitat is the physical place or environment in which it lives. As each habitat has a specific range or regime of abiotic factors the tolerance levels of a species (to abiotic factors) controls </p>
							<h4> 2. Niche: Feeding Relationships (Biotic Factors)</h4>
								<p> Organisms are also restricted by their relationships with other organisms in either habitat. These relationships, and their tolerance to them, therefore, structures their niche.
e.g., producers (autotrophs), consumers (heterotrophs) and detritivores.
</p>
							<h4> 3. Niche: Interaction with other species (Biotic Factors)</h4>
								<p> Interactions (which we have defined 3.1.2 i)</p>
								<ul>
									<li> Competition </li>
										<li> Predation </li>
										<li> Disease</li>
										<li> Symbiosis;</li>
											<ul>
												<li> Commensalism</li>
												<li> Parasitism</li>
												<li> Mutualism</li>
												<li> Amensalism </li>
											</ul>
								</ul>

								<p> For the organisms this means
								<dl><dt>Competition:</dt>
									<dd><li> Availability of food or prey </li>
									<li> Number and variety of competitors </li></dd>
									<dt> Predation </dt>
										<dd><li> Number and variety of predators </li></dd>
									<dt> Disease </dt>
										<dd><Li> Number, variety and intensity of disease-causing organism </li></dd>
									<dt> Symbiosis </dt>
										<dd><li> Availability of significant symbiotic organism </li></dd>
									</dl>
								</p>
					</div>

							<h3> &#10132; Competition </h3>
							<div class="super-sub-topic">
							<p> Competition describes the active demand of two or more organisms for resource. Each competitor is inhibited in some way by the interaction therefore effecting the realized niche.
							</p>
							<strong class="cogverb" id="3.2.1F"> Understand <span class="tooltiptext"> 3.2.1F: understand the competitive exclusion principle </span></strong>
								<h4> Gause's Principle </h4>
								<p> Gause’s competitive exclusion principle states:<b> 'two or more resource-limited species, having identical patterns of resource use, cannot coexist in a stable environment: one species will be better adapted and will out-compete or otherwise eliminate the other(s)'.</b></p>
								<img src="bio photos/comp.png" width=70%>
								<strong class="cogverb" id="3.2.1G"> Analyse <span class="tooltiptext"> 3.2.1G: analyse data to identify species (including microorganisms) or populations
occupying an ecological niche </span></strong>


				</div>
				<h3> &#10132;Keystone Species </h3>
				<div class="super-sub-topic">
				<strong class="cogverb" id="3.2.1H"> Define <span class="tooltiptext"> 3.2.1H: define keystone species and understand the critical role they play in
maintaining the structure of a community </span></strong> Keystone species.
				<p> A plant or animal that plays a unique and crucial role in the way an ecosystem function.
				</p>
				<ul>
					<li> Maintaining biodiversity (through ecological services)
					</li><li> Controls populations (controlling dominant species </li>
					<li> Provides critical resources for a wide range of species </li>
					<li> Maintains balance of ecological relationships </li>
				</ul>
					<h5> Examples of Keystone Species </h5>
					<p> On the Great Barrier Reef, for example, the <b>corals </b>themselves are significant in maintaining the ecosystem as they not only form part of the food chain but provide myriad microhabitats for other organisms through construction of their hard, skeletal structures. </p>
					<p> Some keystone species are mutualists, for example <b>flying foxes,</b> which are migratory, nomadic mammals that are significant in tree pollination and seed dispersal. </p>
				<button><a href="bio docs\3.2.1 h,i - review.docx"> Review Questions for Keystone Species</a> </button>
				<button> <a href="bio docs\3.2.1 h,i - exitticket.docx"> Exit Ticket for Keystone Species </a> </button>
				<h4> Analysing data around Keystone Species </h4>
				<strong class="cogverb" id="3.2.1I"> Analyse <span class="tooltiptext"> 3.2.1I: analyse data (from an Australian ecosystem) to identify a keystone species
and predict the outcomes of removing the species from an ecosystem. </span></strong>
					<P> Data may be in the form of a food web </p>

					<h4><b> Example Questions </b></h4>
					<div class="example">
					<p> <img src="bio photos/foodweba.png" width=40% class="float-right">
					a) Identify a potential keystone species in this food web. Briefly justify your choice. (3)</p>
						<div class="answer">
						The squid. It is a food source for four different species and is one of three consumers of zooplankton. It has the largest number of interactions of all organisms in this food web. The loss of the squid could easily result in a series of trophic effects that completely change the balance in this system and may see the local extinction of more than one other species.
						</div>

					<p> b) Predict two possible consequences of the removal from this ecosystem of the keystone species that you identified. (2)</p>
						<div class="answer">
							<ul>
								<li> Both lantern fish and sunfish used squid as a food source so both species will now rely more heavily on zooplankton and phytoplankton for food, which may lead to increased competition between sunfish and lantern fish and thus lead to reduction in one or both of these species. </li>
								<li> As squid also fed on zooplankton and phytoplankton, the loss of the squid may result in more food for the sunfish and the lantern fish, resulting in less competition between them and an increase in numbers of one or both.</li>
								<li> Tuna increases predation of sunfish, which becomes the only food source for tuna, leading to a collapse of the sunfish population, followed by a collapse in the tuna population as they eat out their food supply.</li>
								<li> Loss of the tuna could then result in a decrease in the shark and/or marlin populations as they compete for the remaining food source, mackerel.</li>
								<li> Following from the reduction in mackerel, there may be a subsequent increase in the number of lantern fish.</li>
							</ul>
						</div>
					</div>

					<div class="example">
					<p> <img src="bio photos/biotaweb.png" class="float-right" width=55%>Analyse the given information to identify the keystone species. (1) </p>
						<div class="answer">
							Southern Cassowary
						</div>

					<p> Predict the of removing the keystone species from the ecosystem. Give a reason. (2) </p>
						<div class="answer">
							Ecosystem collapse.  Cassowaries consume, disperse and help germinate the seeds of significant fruiting trees; the loss of cassowaries would effectively prevent germination of these important trees. As a result, forest diversity would be significantly reduced and other consumers reliant on these species would consequently be affected.
						</div>

					</div>
				</div>

		</div>


		<h2> Population Ecology </h2>

		<div class="content">



		<h3 id="limit"> &#10132; Limiting Factors </h3>
			<div class="super-sub-topic">
        <h4> Features of Populations </h4>
        <p> <b> Population size: </b> the total number of organisms in the population (n).<br>
        <b>Population density: </b>the number of organisms per unit area (n/area).
        </p>

				<p> A limiting factor is an interaction or a resource that constrains a population's size and slows or stops it from growing. These both control the maximum size of the population and the speed of change of the population.</p>
				<table>
					<tr>
						<th> Density Dependent (generally biotic factors) </th>
						<th> Density Independent (generally abiotic factors) </th>
					</tr>
					<tr>
						<td>
							<ul>
								<li> Disease </li>
								<li> Competition </li>
								<li> Predation </li>
							</ul>
							Factors that have an increasing effect on population growth (negative or positive) as the density of the population increases.
						</td>
						<td>
							<ul>
								<li> Temperature </li>
								<li> Moisture </li>
								<li> Acidity</li>
								<li> Availability of nutrients </li>
								<li> Habitat disruption (natural disaster </li>
							</tr>
							</ul>
							Extreme climatic events (drought, cyclones, global temperature change)
Factors that have a controlling effect on population size and growth, regardless of the population density.
						</td>
					</table>

					</div> <!-- for limiting factors -->

		<h3> &#10132; Carrying Capacity </h3>
			<div class="super-sub-topic">
					<p> <strong class="cogverb" id="3.2.2A"> Define <span class="tooltiptext"> 3.2.2A: define the term carrying capacity</span></strong> Carrying capacity:
					<br>the size of the population that can be supported indefinitely on the available resources of that ecosystem. Referred to as ‘K’.
					</p>
					<p>populations will tend to grow exponentially as long as there are resources available (no limitations). This is prevented by <b> environmental resistance </b>(the sum total of all environmental limiting abiotic and biotic factors)
					</p>
					<p>  <strong class="cogverb" id="3.2.2B"> Explain <span class="tooltiptext"> 3.2.2B: explain why the carrying capacity of a population is determined by limiting
factors (biotic and abiotic) </span></strong> Carrying capacity:
					<br> . Because biotic factors are density dependent, as a population’s size increases and density increases, individuals become more crowded and availability of resources becomes restricted, organisms become more susceptible to factors such as competition, predation and disease.
					As a result, the sum of the limiting factors (both abiotic and biotic) increases as the population nears the carrying capacity, births decline, and death rises. Eventually because environmental resistance is so great, birth=death so the population stops growing.
					</p>
					<img src="bio photos/carrying.png" width=50%>
					<strong class="cogverb" id="3.2.2F"> Discuss <span class="tooltiptext"> 3.2.2F: discuss the effect of changes within population-limiting factors on the carrying
  capacity of the ecosystem </span></strong> The effect of changes within population limiting factors on carrying capacity.
						<div class="example"> Moroccan Mussels
							<p> Which limiting factor is likely to have the greatest impact on the carrying capacity of this species?

							</p>

							<div class="answer">
							Space
							</div>

							<p> Justify your response.
							</p>
							<div class="answer">
							Being sessile organisms, they need solid substrate to fasten on to and in highly dense populations space can be a limiting factor. This might not be the case if population are kept low by predation or food access.
							</div>
							</div>

							<div class="example"> Effect of limiting factors: Tiger Quolls
							<p> Which limiting factor is likely to have the greatest negative impact on the carrying capacity of this species?
								</p><div class="answer">
						 Prey (food) availability
							</div>
								<p>Justify your response.</p>
								<div class="answer">
							Organisms with large ranges that are high up on the trophic pyramid are often limited by food densities. This explains their low density and large territories.
									</div>

							</div>
							</div> <!-- for carrying capacity -->

		<h3> &#10132; Population Growth: Lincoln Index: </h3>
			<div class="super-sub-topic">
					<h4> Population Dynamics </h4>
          <strong class="cogverb" id="3.2.2D"> Use <span class="tooltiptext"> 3.2.2D: Use the Lincoln Index to estimate population size from secondary or primary data
           </span></strong>
						Key factors for study include:
							<ul>
								<dt> Population Growth Rate </dt>
									<li> the  change in the total population size per unit time. </li>
								<dt> Natality (birth rate) </dt>
									<li> the number of individuals born per unit time.</li>
								<dt>Mortality (death rate): </dt>
									<li>the number of individuals dying per unit time.</li>
								<dt>Immigration: the number</dt>
									<li> moving into the population. </li>
								<dt>Emigration:</dt>
									<li>the number moving out of the population. </li>
							</ul>
						<h4> Population Growth Rate </h4>
							<strong class="cogverb" id="3.2.2C">Calculate <span class="tooltiptext"> 3.2.2C: calculate population growth rate and change (using birth, death, immigration
and emigration data) </span></strong>
								<p> The speed that a population changes are called the population growth rate. The rate of a population’s increase or decrease is a measure of how fast the change in population is occurring (r). It can be calculated by:
									<br>r = (b + i ) − (d + e)
									<br>where b = number of births, i = number of immigrants, d = number of deaths and e = number of emigrants.
								</p>
						<h4> Lincoln Index: Capture recapture </h4>
							<p>  Population density estimates of larger or more mobile animals can be made using the capture–recapture (or capture-mark-recapture) method</p>

								<h5> Lincoln Index </h5>
								 <p> Size of population (N) = <sup> M x n </sup> &frasl; <sub> m</sub>
									<br> where M = number originally marked, n = total number captured in the second sample and
									m = number of recaptured (marked) individuals in the second sample.
									population, and the total population can then be estimated using the second sample. </br>
								</p>




              <div class="example">
								<p> The table below shows mark and recapture data for a population of Northern Bettong (Bettongia tropica) found around the Tinaroo Creek are of Far North Queensland. Using Lincoln index, calculate a population estimate for the region as of the second annual survey. (2 marks) </p>

								<div class="answer">
								<table>
									<tr >
										<th rowspan="2"> </th>
										<th rowspan="2"> Original individuals captured </th>

											<th colspan="2"> Second Annual survey </th>


									</tr>
                  <tr>
                  <th> Total captured </th>
                  <th> Tagged individuals captured </th>
                  </tr>
									<tr>
										<td> Number of individuals </td>
										<td> 15 </td>
										<td> 20 </td>
										<td> 10 </td>
									</tr>
								</table>
								<p> ANSWER:
<br> =<sup> (M x n)</sup> &frasl; <sub> m </sub>
<br> = <sup>(15 x 20)</sup> &frasl; <sub> 10 </sub>
<br> =  30</p>
</div>
</div>



							</div> <!-- for Population growth Lincoln index -->


              		<h3> &#10132; Population Growth: Growth Modes  </h3>
              			<div class="super-sub-topic">
                      <strong class="cogverb" id="3.2.2E">Analyse <span class="tooltiptext"> 3.2.2E: Analyse population growth data to determine the mode (exponential growth J-curve, logistic growth S-curve) of population growth </span></strong>
                  <p> Idealized models describe two kinds of population growth forms found in nature: <ul> <li>
            Exponential Growth</li><li>
            Logistic Growth</li> </ul>
            </p>

            <table>
              <tr> <th style="width:50%"> Exponential (J-curve)</th><th> Logistic (S-curve)</td>
              </tr>
              <tr> <td> <img class=" left mini" src="bio photos/j.png" width=40%>
                <p> The Exponential curve (also known as a J-curve) occurs when there is no limit to population size.</p>
                  <p> Colonizing populations may undergo a rapid exponential (logarithmic) increase in numbers to produce a J-shaped growth curve.<br> <br>
          In natural populations, population growth rarely continues to increase at an exponential rate.<br> <br>
          Factors in the environment, such as available food or space, act to slow population growth or even cause a complete crash.
        <br> <B>  Examples: Cockroaches and mice  </b>


              </td>

              <td> <img class="minil left"  src="bio photos/s.png" width=40%>
                <p> The Logistic curve (also known as an S-curve) shows the effect of a limiting factor (in this case the carrying capacity of the environment).
                </p>
                <p> As a population grows, its increase will slow, and it will stabilize at a level (around the carrying capacity) that can supported by the environment.
This type of sigmoidal growth produces the logistic growth curve or S curve</p><p> R and K species growths both fit into logistic growth modes </p>


              </td>
            </tr>

          </table>

<h4> R and K selection </h4>
<p> Two parameters govern the logistic growth of populations.
The intrinsic rate of natural increase or biotic potential. This is the maximum reproductive potential of an organism, symbolized by the letter r.
The saturation density or carrying capacity of the
environment, represented by the letter, K.
</p>
<h6> R selected </h6> <p> Species with a high intrinsic capacity for population increase are called r-selected or opportunistic species
</p>
<h6> K selected </h6>
<p> Species that are K-selected exist under strong competition and are pushed to use available resources more efficiently.

These species have fewer offspring and longer lives.
They put their energy into nurturing their young to reproductive age.
</p>

<h4> Population interactions </h4>
<dl>
  <dt> Competition for Resources </dt>
    <dd> Exploitative competition:  Both organisms competing for the same resource(s).
    When two species compete for one resource with differing capabilities this may result in “competitive exclusion” of one species.
    </dd>
  <dt> Predation (mortality):</dt>
  <dd> Prey population declines when growth rates slow below predation rate (and other mortality terms).
  </dd>
</dl>

                    </div> <!-- population growth: models -->
<h2> Changing Ecosystems </h2>
                    <h3> &#10132; Ecological succession  </h3>
                			<div class="super-sub-topic">
                        <strong class="cogverb" id="3.2.3A"> Explain <span class="tooltiptext"> 3.2.3A: Explain the concept of ecological succession (refer to pioneer and climax communities and seres)
                        </span></strong>

                      <p> <b>Ecological succession </b> is the process by which communities in a particular area change over time. The natural, gradual changes in the types of species that live in an area; can be primary or secondary.
        Succession takes place as a result of complex interactions of biotic and abiotic factors.
        </p>
        <strong class="cogverb" id="3.2.3B">Differentiate <span class="tooltiptext"> 3.2.3B:Differentiate between the two main modes of succession: primary and secondary
        </span></strong>

        <h4> Primary Succession </h4>
        <p> Begins in a place without any soil <ul> <li>
        In the beginning there is only rock, sand, volcanic ash.</li> <li>
        Since there is no soil, there is no community.  </li> </ul>
        </p>

        <h5> 1. First Stage of primary Succession </h5>
        <p>  Colonisation of this bare surface is by hardy autotrophs (<b> pioneer species </b>). As lichens age and die, decomposing bacteria break down small amounts of organic matter that collect in rock cracks, which begins the formation of soil.
        </p>

        <h5> 2. Progressive shift between sere </h5>
        <p> The build-up of soil makes these places suitable for mosses. Further soil enrichment is provided by the mosses as they in turn die. Over time, more complex plants are able to grow, progressing from ferns to grasses, to shrubs, to small trees and finally a forest. At each successive change in vegetation type, the replacement plants are better able to compete for resources than their predecessors and new transitional communities referred to as ‘sere’ are formed. </p>

        <h5> 3. The Climax community </h5>
        <p> Eventually, a relatively stable, complex community it reached. A climax community is a mature, stable community that is the final stage of ecological succession
        </p>

        <strong class="cogverb" id="3.2.3C"> Identify <span class="tooltiptext"> 3.2.3C: Identify the features of pioneer species (ability to fixate nitrogen, tolerance to extreme conditions, rapid germination of seeds, ability to photosynthesize) that make them effective colonisers
        </span></strong>

        <h6> Pioneer Species </h6>
        <p> These are hardy species which are the first to colonize barren environments or previously biodiverse steady-state ecosystems that have been disrupted. have one or more of the following features:
                  <ul> <LI> are tolerant of extreme conditions </li> <li>
         are able to photosynthesize </li> <li>
         have rapid spore or seed germination </li> <li>
         use wind pollination and/or dispersal of spores or seeds </li> <li>
         have the ability to fix nitrogen from the air </li> <li>
         are opportunists or r-strategists that disperse rapidly, have a large number of offspring and short life cycle.
        </li> </ul >

        <h4> Secondary Succession </h4>
        <p> Secondary succession is the ‘second time’ the area has been colonised. It is defined as ‘successive, natural changes in plant communities in an area where a previous community has been removed’.
        </p>
      <p> In general this means that: Organisms are destroyed but the soil is safe.  The soil already contains the seeds of weeds, grasses, and trees. Succession begins again but the primary species are different. During secondary succession the ecosystem still experiences the transition between pioneer species, multiple seres and climax community, however, because soil is present, this succession is faster.
Forests formed as a result of secondary succession are called secondary forests.</p>


</div> <!--div for ecological succession-->

		<h3> &#10132; Fossils </h3>
			<div class="super-sub-topic">


			<strong class="cogverb" id="3.2.3D"> Analyse <span class="tooltiptext"> 3.2.3D: analyse data from the fossil record to observe past ecosystems and changes
in biotic and abiotic components </strong>
			<p> changes in climate also affected the different species of plants and animals that were able to grow and reproduce during that time. The fossilised remains or traces of organisms that existed during that period can also provide an indication of how the ecosystem has changed.
			</p>
			<p> Fossils are most commonly found in areas where, on death, their remains are protected from predatory consumption or bacterial decomposition. Generally, the further the fossil is found from the surface, the older it is.</p>

		<h4> Limitations in using Fossil Evidence </h4>
			Limitations include:
		<ul>
			<li> not all biota are preserved as fossils (soft bodied or naturally rare often don’t turn up).
			</li><li>two species that may look similar may not be distinguishable in the fossil record.
			</li><li>the fossils that are preserved are not a representative sample (ratios of organisms found are often not representative).
			</li>
		</ul>

		<h4> Menhinick’s index: Measurements of relative Species Richness in fossil strata. </h4>
			<p> species richness is known as D. This allows us to correct the calculated result for potential bias that effects the number of available samples, such as the effect of time and geological processes on the fossil record. The Menhinick's index. </p>
			 <img src="bio photos/mech.png"> <p>where s equals the number of different species represented in your sample, and N equals the total number of individual organisms in your sample.
			</p>

				<div class="example">
				<img src="bio photos/fossils.png" width=50%>
					<p> From the given information, determine the species richness for strata 1. Use the following formula:  </p>

					<div class="answer">
					<img src="bio photos/d=.png">
					</div>
					</div>
				<div class="example">
						<p> Contrast this species richness with that of strata 4</p>

					<div class="answer">
					<img src="bio photos/d=c.png" width=70%>
					</div>
					</div>
		</div> <!-- div for fossil fuels super-sub-topic-->

		<h3> &#10132; Successful Change </h3>
			<div class="super-sub-topic">
			<strong class="cogverb" id="3.2.3E"> Analyse <span class="tooltiptext"> 3.2.3E: analyse ecological data to predict temporal and spatial successional changes
        <b><p>Analyse ecological data to predict temporal and spatial successional change.
        </p>
    <table>
      <tr>
        <th> Part A</th>
        <th> Part B </th>
      </tr>
      <tr>
        <td>
          <ol>
            <li> Identify a trend pattern or relationship in the data
              <li> State your response </li>
            </li>
            <li> Draw a conclusion. Describe the cause for the TPR </li>
          </ol>
        </td>
        <td>
          <ol><li> Give an expected result in the future- based on the data. </li>
            <li> Give evidence from the data. Perhaps a reference number </li>
          </ol>
        </td>
        </tr>
    </table></B></span></strong>




			<H4> Predicting temporal and spatial changes in ecosystems </h4>
			<p> The nature of secondary succession depends on the original community and the nature of the disturbance. That is to say that the effect of a disturbance such as fire in a wet sclerophyll forest is completely different to its effect in a dry sclerophyll forest.
			Succession is affected by the plants that invade or grow after a piece of land experiences disturbance. Successional changes in the producer communities can have flow on effects on the consumer communities, their densities and relationships.
			</p>

			<div class="example">
				<p> The table below shows the probability of the replacement of one individual tree by another of the same or different species in 100 years time after a sudden ecological disturbance. </p>
					<img src="bio photos/occupant.png">
					<p> 1. Which species is most likely to replace Acacia sophorae in a 100-year temporal successional change in this forest? [1 mark]</p>
            	<div class="answer">
                Black Butt </div>
            <p> 2. Which species is most likely to replace Eucalyptus tereticornis in a 100-year temporal successional change in this forest? [1 mark]</p>
              	<div class="answer">
                 Blue Gum
                </div>
            <p> 3. Identify the species likely to be the equilibrium species at the end of the successional process. Justify your response. [2 marks]
					</P>
			<div class="answer">
			  Red Mahogany because its likely hood of occupying the space in the next 100 years is 96%
			</div>
					</div>
			<div class="example">
				<img src="bio photos/predict.png" width=50%>
				<p> When predicting successional change, which of the following would typically indicate that an ecosystem is progressing toward its climax community?
					</P>
				<div class="answer">
				 C - number of individuals increase when population increases and hence biomass also does.

					</div>
					</div>

			<div class="example">
				<img src="bio photos/bushrat.png" width=50%>
				<p> Draw three conclusions about the successional relationships in this ecosystem. [3 marks]

        <div class="answer">
       a) The bush rats have very small population as seem by the magnitude of the bar graphs as it did not get above 20 across the years. Therefore, it can be concluded that the bush rats were not adapted to the ecosystem after fire and had low niche accommodation.
          The house rats had high fluctuations in frequency as they had the highest frequency in the 2nd year however, at the 5th year it was the second lowest populated species. This suggests that house mouses are k-selected species.
          The brown antechinus had a u shaped fluctuation between the years with a low point 3 years after the fire and increased in population size before and after the fire.
      </div>
				<p>Predict which mammal will be the most abundant in the sixth year after the bushfire. Give a reason. [2 marks]
				</p>
				<div class="answer">
			 	b) The brown antechinus will most likely have the greatest population numbers as since year 3, B.A showed an increasing trend on population numbers as it grew from about 23 to 58 from 3 to 5 years. This can  be extrapolated to year six where this would also show an increase in population.

					</div>
					</div>
<br><BR>
				<button><a href="bio docs\POWERPOINT - 3.2.3 e - Successional Changes(2).pptx"> See PowerPoint for exit tickets</a> </button>

		</div> <!-- div for successful something super-sub-topic-->

		<h3> &#10132; Human Impacts </h3>
			<div class="super-sub-topic">
			<h5> Definitions </h5>
			<dl>
				<dt> Magnitude </dt>
					<dd> The size or extent of the effect of a factor </dd>
				<dt> Duration </dt>
					<dd> The time in which a factor continues </dd>
				<dt> Speed of change </dt>
					<dd> The rate of change in an environment </dd>
			</dl>
      <strong class="cogverb" id="3.2.3F"> Predict <span class="tooltiptext"> 3.2.3F: predict the impact of human activity on the reduction of biodiversity and on the
magnitude, duration and speed of ecosystem change.  </span></strong>
			<h4> Human impacts on ecosystems </h4>
			<dl>
				<dt>Habitat change:</dt> <dd>loss of habitat through destruction, degradation and fragmentation. Results in reduced populations and diversity of species across habitats.
				</dd><dt>Overexploitation:</dt><dd> ecosystems or resources/species. Results in reduced populations and endangerment of species that are overharvested.
				</dd><dt>Pollution:</dt><dd> Results in air and water pollution that reduced productivity of ecosystems and enhances processes such as global warming.
				</dd><dt>Introduction of invasive species (e.g., toads and rabbits): </dt><dd> Interferes with ecosystem processes and stability and reduces diversity of species across habitats.
				</dd><dt>Climate change:</dt><dd> across local or global scales. Results in changes in ecosystem variables and dynamics, endangering endemic species and reducing the diversity and stability of ecosystems.
				</dd>
			</dl>
			<p>
				Each of these may impact on a; local through to global scale and may vary in the speed and duration of their effect on the ecosystems or organisms in question.
			</p>


		</div> <!-- div for human impacts super-sub-topic-->


		<!--content-->
		</div>




		<!-- Unit 4 starter                        -->




		<div id="unit4" class="topic">
	<h1> Unit 4 Topic 1: DNA </h1>
</div>
		<div class="sub-topic">
		<h2> DNA structure and replication </h2>
	</div>
		<div class="content">
		<h3> &#10132; DNA </h3>
				<div class="super-sub-topic">
	<strong class="cogverb" id="4.1.1A"> Understand <span class="tooltiptext"> 4.1.1A: understand that deoxyribonucleic acid (DNA) is a double-stranded molecule
that occurs bound to proteins (histones) in chromosomes in the nucleus, and
as unbound circular DNA in the cytosol of prokaryotes, and in the
mitochondria and chloroplasts of eukaryotic cells. </span></strong>
	<button> <a href="bio docs\POWERPOINT - 4.1.1.a,b - DNA Structure.pptx" > Click for Power Point of DNA </a> </button>

		<p> DNA: Deoxyribose Nucleic Acid  <img src="bio photos/dna.png" width=40% class="float-right">
		<ul>
			<li> It can be found in the nucleus of eukaryotes or the cytosol of prokaryotes and organelles such as mitochondria and chloroplasts </li>
			<li> It contains a code unique to the individual that can be passed to offspring from generation to generation with little or no change.
			</li>
		</ul>

		<p> DNA  is a very large molecule made up a long chain on sub-units.
			<br> The subunits are called <b> nucleotides </b>. Each nucleotide is made up of two consistent subunits:
				<ol>
					<li> a sugar called deoxyribose </li>
					<li> a phosphate group PO<sub>4</sub></li>
				<p>And one variable subunit </p>
					<li> a nitrogenous (organic base)</li>
				</ol>
		</p>
		</p>

		<h5> Nitrogenous Bases </h5>
<img src="bio photos/base.png" width=20% class="left">
		<p>The most common nitrogenous bases are
		<ul>

			<li> Adenine </li>
			<li> Thymine </li>
			<li> Cytosine </li>
			<li> Guanine </li>
		</ul>

		</p>
"
		<p> The phosphate of one nucleotide forms a strong covalent bond with the sugar ring. (deoxyribose) of the next nucleotide. Producing a "backbone" to the molecule. <b> THINK: Sugar phosphate back bone, nitrogenous base </b>
		</p>
		<strong class="cogverb" id="4.1.1B"> Recall  <span class="tooltiptext"> 4.1.1B: recall the structure of DNA, including:<br>
    <uL>
<li> nucleotide composition</li>
<li>complementary base pairing</li>
<li>weak, base-specific hydrogen bonds between DNA strands</li>
</ul> </span></strong>
		<p> The sugar (deoxyribose)- phosphate chains are on the outside and the two strands <b> are held together by weak base specific hydrogen bonds between the nitrogenous bases</b>.
		</p>

		<h5> Complementary nitrogenous base pairing </h5>
			<p> The bases always pair up in the same way. Adenine forms a bond with Thymine.</p>
			<img src="bio photos/adenine.png" >

			<p> and Cytosine bonds with Guanine		</p>
			<img src="bio photos/cytosine.png">


		<h4> DNA Structure </h4>
			<p> The order of the bases in one strand determines the order of the bases in the other strand. <br> <br> The paired nitrogenous bases are chemically bonded together using weak chemical bonds called hydrogen bonds. These bonds are base-specific resulting in the observed base pairing.
			</P>

		<img src="bio photos/dnastructure.png" width=40% class="right">

		<p> Paired strands are coiled into a spiral called a <b>double helix</b> </P>
		 <img src="bio photos/double.png" width=30% class="right">
     <br> <br> <BR> <br> <br> <BR> <br> <br> <BR> <br> <br> <BR>
     </div> <!-- div for DNA-->

		<h3> &#10132; Chromosomes </h3>
				<div class="super-sub-topic">
				<p> Chromosomes are stands are circular or linear genetic information located in the nuclei or cytosol of cells. </p>

			<h5> Prokaryotic DNA</h5>
			<p> Prokaryotes have one chromosome consisting of a single circular DNA molecule (chromosome). This DNA in not bound by a nuclear membrane or to histone packing proteins but can be found in the cytosol of the cell.
			<br> <br> Eukaryotic organelles such as mitochondria and chloroplasts have their own similarly unbound circular DNA.
			</p>

			<h5> Eukaryotic DNA</h5>
				<p> Eukaryote chromosomes are linear DNA molecules. They are bound by a membrane to form the nucleus of the cell.
				To organise the DNA and prevent it from tangling each strand is bound to a group of 8 proteins called <b>histones</b>. The DNA-histone complex that is formed is called a <b>nucleosome.</b> Because DNA wraps around histones, they also play a role in gene regulation.
				</p>

				<h6> Diagram of a Nucleosome. Note the 8 histone molecules and the DNA wrapped around it </h6>
					<img src="bio photos/histone.png">


				</div> <!-- div for chromosomes-->

		<h3> &#10132; DNA Replication </h3>
				<div class="super-sub-topic">
				<strong class="cogverb" id="4.1.1C"> Recall  <span class="tooltiptext"> 4.1.1C: explain the role of helicase (in terms of unwinding the double helix and
separation of the strands) and DNA polymerase (in terms of formation of the
new complementary strands) in the process of DNA replication. Reference
should be made to the direction of replication. </span></strong> <button> <a href="bio docs\POWERPOINT - 4.1.1.c - DNA Replication(3).pptx"> Click for PowerPoint on Replication </a> </button>
				<p> DNA replication is termed <b>semi-conservative </b> replication because each replicated DNA contains one strand of the original DNA and one new strand. </p>

				<h4> DNA Directionality </h4>

				<img src="bio photos/replication.png" width=50% class="left">

				<p> The two DNA strands have directionality. Each strand has a 3' (three-prime) end. The opposite end of each strand ends in a 5'(five-prime) end. Each nucleotide on the complementary strand is reversed. The strands of the double helix therefore run from <b> 3 to 5 on the left </b> and from <b> 5 to 3 on the right </b>. They are antiparallel. This directionality is important in DNA replication because the enzyme DNA polymerase can only synthesis DNA in one direction. <b> Polymerase reads DNA from 3' to 5’ but builds in 5' prime to 3'.</b>

				</p>

				<br><BR> <BR><br><BR>
				<h5> Separation: HELICASE </h5>
        <img src="bio photos/35.jpg" width=35% class="right">
					<p> DNA must first be unwound and unzipped by an enzyme called DNA helicase. The enzyme unwinds then separates the two strands of parent DNA by breaking the hydrogen bonds between the nitrogenous bases. This results in the exposure of the two strands of nucleotide in the formation called the replication fork.
					</p><br><BR><BR><BR><BR><BR>
				<h5> Elongation: DNA POLYMERASE </h5>
					<p> The enzyme DNA polymerase moves along the exposed nucleotides generating new DNA strands that are complementary to the ‘original’ template strands, added to the 3’ end.
 The DNA polymerase then proofreads the newly synthesised strands of DNA</h5>

				<h5> Elongation Direction </h5>
				<p> The DNA strand is synthesised in a 5’ to 3’ direction (with new bases added to the 3’ end of the new strand) with the polymerase moving 3’ to 5’ along the strand it is reading. Thus, the nucleotides are assembled in a continuous fashion on one strand (called the ‘Leading Strand’) but in short fragments on the other strand (called the ‘Lagging Strand’).</p>

				</div> <!-- for replication-->

	</div> <!-- div for table division 1 for unit 4 DNA replication -->

	<div class="sub-topic">
		<h2> Cellular Replication and Variation </h2>
		</div>

		<div class="content">
				<h3> &#10132; Cellular Replication: Meiosis </h3>
				<div class="super-sub-topic">
			<strong class="cogverb" id="4.1.2A"> compare and contrast <span class="tooltiptext"> 4.1.2A: compare and contrast the process of meiosis I and II
­ recognise the role of homologous chromosomes
­ describe the processes of crossing over and recombination and
demonstrate how they contribute to genetic variation
­ compare and contrast the process of spermatogenesis and oogenesis
(With reference to haploid and diploid cells). </span> </strong> <button> <a href="bio docs\POWERPOINT - 4.1.2. a - Cellular Replication; Meiosis (5).pptx" > PowerPoint for Cellular Replication </a> </button>
			<button> <a href="https://quizizz.com/admin/quiz/5e955a2fd5fe26001bb9c3ba"> Quizziz for Cellular Variation and Gametogenesis</a> </button>
			<h4> Homologous Chromosomes </h4>
				<p>
				Chromosomes are matched in pairs based on their size and shape. (One from female parent and another from male parent). Each pair contains one chromosome inherited from each parent.
				</p>
				<p> We have 23 sets of Homologous Chromosomes for a total of 46. </p>


				<h5> Replication </h5>
				<p> There are two forms of replication :
					<dl>
						<dt> Mitosis:</dt>
							<dd>
								<ul>
									<li> Division used for growth </li>
									<li> Used for growth and repair </li>
								</ul>
							</dd>
						<dt> Meiosis: </dt>
							<dd>
								<ul>
									<li> Division for gametes (sex cells)</li>
									<li> Used solely for sexual reproduction </li>
								</ul>
							</dd>
					</dl>
				<h4> Mitosis </h4>
				<p><img src="bio photos/meiosis.png" width=20% class="left mini"> </p>
				<dl>
					<dt> INTERPHASE</dt>
						<dd> genetic material doubles </dd>
					<dt> PROPHASE </dt>
						<dd>Chromosomes pair up. 2 chromatids joined by a centromere. centrioles move to opposite sides of nucleus</dd>
					<dt> METAPHASE</dt>
						<dd> chromosomes meet in middle </dd>
					<dt> ANAPHASE</dt>
						<dd> chromosomes pulled apart </dd>
					<dt> TELEPHASE</dt>
						<dd> Finally there are two cells. Chromosomes uncoil and spindle fibers disintegrate.  </dd>
					</DL>

				<h4> Meiosis </h4>
        <img src="bio photos/meiosis.stages.jpeg" width=40% class="left">
					<p> 2 sets of cell division involved. 4 daughter cells produced. Each daughter cell has half the chromosome of the parent, with one of each pair of homologous chromosomes in each. <br><br> This halving of chromosomes produces a cell that is described as being haploid (half the total number of chromosomes) and is represent as ‘n’. This is in contrast to 2n in somatic, diploid cells with the full complement of 46 chromosomes
					</p>

					<h6> Stages of Meiosis </h6>
						<p> homologous chromosomes side by side in ‘Metaphase 1’ (rather then above each other as seen in Mitosis). This a</p>
							<ol>
								<li> The role of homologous chromosomes in meiosis one is to provide opportunity for <b>crossing over</b> <b> (prophase II) </b>and <b>independent assortment (metaphase II)</b> and therefore recombination of genes, increasing diversity. The homologous chromosomes then separate in Anaphase 1.
								</li><li>The role of these homologous chromosomes in meiosis two is the separation of sister chromatids. This means that by prophase 2 each daughter cell (gametes) is haploid and has only 23 chromosomes.
								</li>
							</ol>


		</div> <!-- div for cellular replication meiosis -->


		<h3> &#10132; Cellular Variation </h3>
				<div class="super-sub-topic">
			<strong class="cogverb" id="crossingover"> compare and contrast <span class="tooltiptext"> 4.1.2A: compare and contrast the process of meiosis I and II
­ recognise the role of homologous chromosomes
­ describe the processes of crossing over and recombination and
demonstrate how they contribute to genetic variation
­ compare and contrast the process of spermatogenesis and oogenesis
(With reference to haploid and diploid cells).A </span> </strong>

			<h5> Genes </h5>
				<p> Specific segment of DNA/chromosomes that code for a specific protein or RNA.
A single chromosome has hundreds of genes.
The instructions in genes allow them to make proteins – which are the building bocks of life.
</P>

<p> <b> LOCUS: </b> Location of Genes. <br> <br> The position of a gene on a chromosome is the locus. Chromosomes from a homologous pair have genes that control the same trait at the same locus. </p>
<p> <b> ALLELES </B> Genes occupying the same position (locus) on homologous chromosomes are called alleles. </p>
<ul>
	<LI> Alleles are versions of the same genes that code for a variant (version) of the same polypeptide or protein </li>
	<li> Any one individual can only have a maximum of two alleles for a given gene.</li>
<li> There may be more than two alleles in a population, e.g. blood groups A, B, O.</li>
</ul>

<h4> Increasing Genetic Variation </h4>
	<p> <img src="bio photos/variation.png" class="left">	 The <b>law of segregation </b> states that during the production of gametes, the two copies of each hereditary factor segregate so that offspring acquire one factor from each parent.
		</p>
    <strong class="cogverb" id="4.1.2B"> compare and contrast <span class="tooltiptext"> 4.1.2B:demonstrate how the process of independent assortment and random
fertilisation alter the variations in the genotype of offspring.  </span> </strong>
	<p> The <b> law of independent assortment </b> states that when two or more characteristics are inherited, individual genes (or the homologous chromosomes on which they are found) assort independently of other genes (or the homologous chromosome pairs) during gamete production, giving different traits an equal opportunity to occur together.
	</p>

	<p> The laws of, segregation and independent assortment, mean that during meiosis each homologous chromosome in a pair orientates into one of two groups of chromosomes independently relative to all other pairs of homologous chromosomes. This occurs during Metaphase 1 of Meiosis, when the homologous chromosomes pair up next to each other.
 Thus, a gamete receives 23 chromosomes that are made up of chromosomes from both the paternal and maternal sets of chromosomes (despite receiving only one of each homologous chromosome). This process greatly increases the variety of possible gene combinations possible in any fertilised egg, as it will be a combination of two sets of recombined chromosomes. </p>

<h5> Crossing Over </h5>
<p> <img src="bio photos/cross.png" class="left">
	 <b> Crossing Over </b> occurs when there is an exchange of segments of DNA between homologous chromatids during prophase I of meiosis. It may occur at one or more places along the chromosome.
			Adjacent segments of chromatids from the homologous chromosomes break off and rejoin with the partner chromatid. Crossing over has survival value for the individual since some of the gametes have combinations of linked genes that are different from the parental combination.

<h5> Variation </h5>
 <p> The processes of segregation, independent assortment and crossing over have the possibility of giving rise to a vast array of different allele combinations in the gametes produced. The more chromosome pairs there are in a cell, the more variation is possible. As the process of fertilisation is totally random in natural conditions, it is impossible for sexually reproducing organisms to produce an identical clone to either parent. The main advantage of sexual reproduction is that it gives rise to many variations in the genotype of the offspring upon which natural selection may act.
</p>

			</div> <!-- div for variation -->

				<h3 id="sperm"> &#10132; Gametogenesis </h3>
				<div class="super-sub-topic">

			<div>
				<h4> Spermatogenesis </h4> <img src="bio photos/sperm.png" width=25% class="right">
				<p> Spermatogenesis is the production of spermatozoa (sperm). At puberty (the onset of sexual maturity) in males, this process starts.
				</p>
				<h6> Stages of Spermatogenesis </h6>
				<p>

					<ul>
						<li> Germ cells called <b> spermatogonia </b> (2n) divide endlessly by mitosis to produce more spermatogonia (2n).
						</li><li>One of each two grows into one larger cell called <b> Primary spermatocytes </b>.
						</li><li>At the end of Meiosis 1, each primary spermatocyte produces two secondary spermatocytes.
						</li><li> Each secondary spermatocyte matures through differentiation into spermatozoa (sperm cells).
						</li><li> In males, spermatogenesis begins at puberty and continues into old age. </li>
					</ul>
				</p>
			</div>

			<br><br><br><br><br><br><br><br><br><br>
				<h4> Oogenesis </h4>
				<img src="bio photos/egg.png" class="left"><p> Oogenesis is the production of an ovum (egg) it takes place in the ovaries. The production and release of a secondary oocyte is cyclic- it usually occurs once each month (approximately every 28 days.)
				<b> Unlike </b> spermatogenesis, females carry their entire complement of oocytes at birth and Egg production eventually becomes irregular and stops at menopause.
				</p>
					<h6> Stages of Oogenesis </h6>
					<p>

							<ul>
								<li> In the ovaries of a female fetus, germ cells called oogonia (@n) divide by mitosis to form more oogonia (2n).
								</li><li>Oogonia grow into larger cells called primary oocytes (2n) where they pause until puberty.
								</li><li>Every menstrual cycle a few primary follicles start to develop (only 1 matures).
								</li><li>The primary oocyte completes the first division of meiosis forming two haploid nuclei (n).
								</li><li> The cytoplasm of the primary oocyte is divided unequally forming a large secondary oocyte (n) and a small polar cell (n)
								</li><li>When the mature follicle bursts, (ovulation)
								</li><li>The second division of meiosis is only completed if a sperm penetrates the secondary oocyte, fertilising it.
								</li><li>The completion of the second division of meiosis also produces a second polar cell.  The first and second polar bodies do not develop and eventually degenerate.
								</li>
						   </ul>
					</p>

				<h5> Similarities between oogenesis and spermatogenesis </h5>
				<ol>
					<Li> Both start with proliferation of cells by mitosis (2n to 2n)</li>
					<li> Both go through the two divisions of meiosis </li>
					<li> Both form haploid (n) gametes through meiosis </li>
					<li> In both cases the original stem cells are diploid  (2N) prior to meiosis.
					</li><li> Both start with haploid (n) and end with diploid (2n). </li>
				</ol>

				<h5> Differences between oogenesis and spermatogenesis </h5><table>
			<tr>
				<th> Spermatogenesis </th>
				<th> Oogenesis </th>
			</tr>
			<tr>
				<td> Millions of spermatozoa </td>
				<td> One produced every 28 days </td>>
			</tr>
			<tr>
				<td> Sperm Formation starts during puberty </td>
				<td> Egg production starts during fetal development.</td>
			</tr>
			<tr>
				<td> Sperm production continues throughout adult life of men.</td>
				<td> Egg production becomes irregular and stops at menopause.</td>
			</tr>
			<tr>
				<td> 4 spermatocytes are produced </td>
				<td> Only results in 1 ovum and three polar bodies (that degenerate).</td>
			</tr>
			<tr>
				<td> Spermatogenesis produces mature sperm in an uninterrupted sequence.</td>
				<td> Oogenesis is not completed without stimulation from a sperm cell</td>
			</tr>
			<tr>
				<td> Sperm contain very little cytoplasm </td>
				<td> The egg must increase its cytoplasm (and organelles)</td>
			</tr>

		</table>

		</div> <!-- div for gametogenesis-->

		</div> <!-- div for second division in topic 2 (cellular replication and variation -->

		<div class="sub-topic">
		<h2> Gene Expression </h2>
		</div>

		<div class="content">
				<h3> &#10132; Gene and Genome </h3>
				<div class="super-sub-topic">

			<button> <a href="https://quizizz.com/admin/quiz/5e9a58311d06860020d644b7"> Quizziz for Gene Expression</a> </button>

			<strong class="cogverb" id="4.1.3A">Define <span class="tooltiptext"> 4.1.3A: Define the terms gene and genome</span> </strong> Gene and Genome:
			<p> A <b> gene </b> is a region/s of DNA that are made up of nucleotides; the molecular unit of heredity'.
			<br>
			<br>
			It may also be described as a specific segment of DNA/ chromosomes that code for a specific protein</p>

			<p> A <b> genome </b> is all the genetic material in the chromosomes of an organism, including its genes and DNA sequences.
			</p>
			<strong class="cogverb" id="4.1.3B"> Understand <span class="tooltiptext"> 4.1.3B: understand that genes include ‘coding’ (exons) and ‘noncoding’ DNA (which includes a variety of transcribed proteins: functional RNA (i.e., tRNA), centromeres, telomeres and introns. Recognise that many functions of ‘noncoding’ DNA are yet to be determined) </span> </strong>
			<h4> Non-coding DNA </h4>
			<p> Genes also include sections of DNA that do not encode for protein sequences referred to as<b> non-coding DNA. </b>. non-coding DNA have a variety of biochemical and structural functions, which include:
			</p>
			<ul>
				<li> transcription of non-coding or functional RNA (such as rRNA [ribosomal RNA], tRNA [transfer RNA] and non-coding RNAs that influence gene expression)
				</li> <li> regulating gene expression. If the segments of non-coding DNA are responsible for controlling structural genes, they are termed regulatory genes.
				</li> <li> adding the end caps, called telomeres, to each chromosome (these provide protection to DNA during replication)
				</li> <li> forming the centromere (points of attachment of chromatids during replication)
				</li> <li> introns. These are non-coding sections of RNA or DNA that interrupts a gene-coding sequence  or are spliced out before the RNA molecule is translated into a protein.  Such as the origin and terminus sequences involved in chromosome replication.
				</li> <li> many of the functions of ‘noncoding’ DNA are yet to be determined
				</li>
			</ul>
			</div> <!-- div for gene and gene expression -->

			<h3> &#10132; Protein Synthesis: Transcription </h3>
				<div class="super-sub-topic">
				<button> <a href="bio docs\POWERPOINT - 4.1.3. a,b,c,d - Protein Synthesis.pptx"> Power Point for Protein Synthesis </a> </button>
				<br><br>  <strong class="cogverb" id="4.1.3C"> Explain <span class="tooltiptext"> 4.1.3C: explain the process of protein synthesis in terms of ­ transcription of a gene into messenger RNA in the nucleus ­ translation of mRNA into an amino acid sequence at the ribosome (refer to transfer RNA, codons and anticodons)</span> </strong>
				<h5> Protein </h5>
				<p> Proteins are organic compounds made up of chains of amino acids.
					There are 20 amino acids commonly used in your body.
					Most of your body structure is made of some type of protein.
					</p>
          <strong class="cogverb" id="4.1.3D"> Recognise <span class="tooltiptext"> 4.1.3D: recognise that the purpose of gene expression is to synthesise a functional
  gene product (protein or functional RNA); that the process can be regulated
  and is used by all known life
           </span> </strong>
				<p> <b> Gene Expression</b> the two–stage processing of information encoded in DNA to produce proteins. The purpose of gene expression is to synthesise a functional gene product (protein or functional RNA); that the process can be regulated and is used by all known life</p>
				<img src="bio photos/central.png">

				<h5> RNA </h5>
				<p> DNA codes for the formation of <b> RNA </b> in the nucleus of the cell.
	RNA is short for <b> ribonucleic acid.</b>   RNA is very similar in structure to DNA except for three small differences.  These are the fact that RNA is a single stranded molecule, and it lacks the base thymine (T) as it is replaced by the base uracil (U), and its five-carbon sugar ribose has one more oxygen atom than the sugar deoxyribose in DNA.
Three different types of RNA exist, mRNA or messenger RNA, tRNA or transfer RNA,
</p>
				<h4> Stages of Protein Synthesis </h4>
				<P>
				<dl> <dt> <b> Transcription </b> </dt>
					<dd><img src="bio photos/trans.png" width=50% class="left">
					is the stage of protein synthesis in which the information in DNA (a section referred to as a gene) for making a protein is transferred (transcribed) to an RNA molecule?
					<ol>
						<li> <b> RNA polymerase </b> binds to a segment of DNA where a certain gene sequence is located, breaking the weak bonds between bases <b>(unzipping) </b> </li>
						<li> As the RNA polymerase moves along the template DNA strand, <b> free RNA nucleotides </b> in the nucleoplasm finds their <b>complementary base pair</b> on the DNA causing bonds to form. RNA nucleotide is complementary to the DNA.</li>
					<p> The mRNA will be used to make a specific protein that was encoded in the DNA.</p>
						<li> <b>RNA polymerase catalyses </B>the formation of chemical bonds between ribose sugar and phosphate groups, <b>joining the RNA nucleotides</b> together. </li>
						<li> <b>Introns of mRNA are cut off</b>, leaving only exons in a process called splicing, creating <b>mature mRNA. </b></li>
						<li> This mRNA is free to pass through the nuclear envelope and into cell cytoplasm where it is met by a ribosome, ready to undergo Translation. </li>

					</ol>
					</p>
					<br>
						<p> <img src="bio photos/condons.png" width=50% class="right">
					<b> Codons </b> are a three-nucleotide sequence in DNA or mRNA that encodes for an amino acid or signifies a stop signal, they are also transcribed from the triplets.
					</P>
					<p> <b> Intron </b> is a noncoding section of DNA within a split gene. Transcribed from eukaryotic DNA but are removed before translation of mRNA into a protein occurs.
					<br><br> <b> Exon </b> is a coding section of DNA within a split gene that gets transcribed and translated </p>

					</dd>
					<br>
					<br>
				<dt> Translation </dt>
					<dd> <img src="bio photos/transl.png" width=70% class="left">
						<ol>
							<li> Once the mRNA is in the cytoplasm, it <b> attaches to a ribosome </b>  where it <b> reads </b> in groups of 3 letters <b> (codons) </b> </li>
							<li> As the ribosome moves to each new codon position, a <b> tRNA </b> carrying its<b> specific amino acid approaches. </B> </li>
							<li> Each<b>  tRNA carries </b> a group of three RNA referred to as an <B>anticodon </B> (generally specific to that amino acid), if the tRNA’s <b> anticodon is complementary </b> to the codon, it <b> links to the mRNA </b>  with weak hydrogen bonds.</li>
							<li> Between <b> two anti codons, a peptide bond </b> is formed then the first <b> tRNA detaches from anti-codon.</b>  The <b> ribosome holds the two amino acids in place </b> so that a <b> peptide bond forms </b> between them. As the ribosome moves along the mRNA, more and more amino acids are added to the <b> growing polypeptide chain. </b>  </li>
							<li> This process is <b> repeated </b> until the ribosome reaches the <b> end of the mRNA,</b>  where it drops off and liberates its polypeptide chain. </li>
							<li> Eventually when the entire mRNA has been translated the <b> polypeptide splits </b>  off inside the RER and is wrapped up to form a protein. This may be exported to elsewhere in the cell or taken to the membrane to be dumped into the extracellular fluid. </li>
						</ol>
					<p> <b> Anticodon </b> is a three-nucleotide sequence on tRNA (transfer RNA) that recognizes a complementary codon on mRNA. </p>

					</div> <!-- div for protein synthesis -->


					<h3> &#10132; Gene Regulation </h3>
				<div class="super-sub-topic">
				<button> <a href="bio docs\POWERPOINT - 4.1.3. e - Control of Gene Expression(5).pptx"> Power Point for Gene Expression and Epigenetics </a> </button>
					 <strong class="cogverb" id="4.1.3E"> Identify <span class="tooltiptext"> 4.1.3E: Identify that there are factors that regulate the phenotypic expression of genes
   <ul> <li> During transcription and translation (proteins that bind to specific DNA
   sequences)</li> <li>
   Through the products of other genes</li> <li>
   Via environmental exposure (consider the twin methodology in epigenetic
   studies)</li> </uL> </span> </strong>

								<p> All the cells in your body carry a copy of your entire genome, but not all cells express (in their phenotype) all the possible genes present in your genome. Instead, cells differentially express genes depending on their needs.
			The phenotypic expression of some genes and not others result in cell differentiation.
			</p>

			<p> The process of expression of different genes at different rates and at different times is referred to as <b> gene regulation. </b>Genes can be regulated at different stages: pre-transcription, post-transcription and translation.
			</p>

			<h5> Pre-Transcriptional Control </h5>
			<p> <b> Regulatory genes </b> associated with the structural gene, or the products of other non-coding sequences, can produce proteins (or functional RNA) that act as <b>transcription factors.</b> These factors (molecules) can bind to specific sequences at the start of a structural gene to either:
			<ul>
				<li>
				block the attachment of RNA polymerase, and so block the expression of that gene
				</li> <li> 	alter (accelerate or decrease) the rate at which the gene is expressed.
				</li> <li> 	Initiate transcription.
				</li>
			</ul>
			</p>

			<p> In some situations, gene expression is altered by the environment by influencing regulatory gene </p>

			<h5> Post-Transcription Control </h5>
				<p> Once the mRNA has been produced, cells are capable of controlling the amount of protein produced. This is achieved by modifying the post-transcriptional process, or longevity of the mRNA.</p>

				<h5> Translation Control </h5>
				<p> Gene expression can be regulated by <b>controlling the length of time the mRNA survives </b>before it is degraded.
				</p> <p> Many different proteins and amino acids are involved in the translation process. By controlling the amounts of these molecules, the rate of gene expression can accelerate or decrease. An example is microRNA.
				</p> <p> A microRNA (abbreviated miRNA) is a small single-stranded non-coding RNA molecule (containing about 22 nucleotides) found in plants, animals and some viruses, that functions in RNA silencing and post-transcriptional regulation of gene expression.
				</p>



					<h4>  Epigenetics </h4>



				<p> The <b> epigenome </b> (‘above the genome’) is a set of factors that affect which part of the DNA is activated.
These factors may occur as a result of intra-cellular or extra-cellular stimuli, by neighbouring cells, by physiology, or entirely by the environment to which the organism is exposed.
</P><p> The epigenome acts by producing chemical tags or factors (methyl or acetyl groups) that do not change the DNA ‘blueprint’, but determine which genes are switched on (expressed) and which are switched off (repressed). These factors bind to specific sequences in the DNA.
</p>
	<p> The activation of these factors can be influenced by a range of variables or interaction . </p>

	<h5> Histone Modification </h5>
	<p> Acetyl group tags the histone structure relaxes, and the nucleosomes move further apart. <b> Gene is switches on (acetyl) </b> . Methyl group tags coiling the chromatin tighter and the nucleosomes closer together. The<b> genes are switched off in those pieces of DNA. (methyl) </b>
	</p>
	</div> <!-- div for epigenetics-->

	<h3> &#10132; Transcription Factors </h3>
				<div class="super-sub-topic">
				<strong class="cogverb" id="4.1.3F"> Recognise <span class="tooltiptext"> 4.1.3F: recognise that differential gene expression, controlled by transcription factors,
regulates cell differentiation for tissue formation and morphology</span> </strong>
				<h4> Morphology and Cell differentiation </h4>
				<p> There are many transcription factors that regulate gene activity (differential expression of genes). Some are produced by non-coding DNA while others are produced by structural genes. They may act to influence the development of the organism. This is done through the control of development of morphology and cell differentiation in tissue formation. </p>
				<p> <b> Morphology, </b> in biology the study of the size, shape and structure of animals, plants and microorganisms and of the relationships of their constituent parts. </p>
				<p> <b> Cell differentiation </b>  is the process in which a cell changes from one cell type to another. Usually, the cell changes to a more specialized type.
				</p>
				<strong class="cogverb" id="4.1.3G"> Recall <span class="tooltiptext"> 4.1.3G: Recall an example of a transcription factor gene that regulates morphology (HOX transcription factor family) and cell differentiation (sex-determining region Y)</span> </strong>
				<h5> HOX Transcription Factor </h5>
				<p> <B> Homeotic genes </b> are genes in the cells that determine which anatomical structures of the body the cells will differentiate into.
The term ‘Hox’ is a contraction of homeobox. These are involved in the regulation of anatomical development in animals, plants and fungi.
</p>
				<p> <b> HOX genes </b> control the body plan of an embryo along the head-tail axis. </p>
				<p> They control the formation of the segmentation of the embryonic body plan. </p>
				<p> the Hox proteins (the products of the Hox genes) act as transcription factors that determine the type of appendages that will form on a segment. Hox proteins control the identity of the body segments through gene regulation, but do not form the actual segments themselves. </P>

				<h5> Cell differentiation: BY the sex-determining region Y </h5>
				<p> The SRY gene is the male sex-determining region that is found on the Y chromosome.
This gene produces a transcription factor that attaches (binds) to specific regions of DNA and helps control the activity of other genes.
It is responsible for the production of the SRY protein, which starts processes that cause a fetus to develop male gonads (testes).
</P>
<p>
Between weeks 9 to 12 of embryonic development there is a peak in the production of these hormones.
This coincides with the differentiation of the male internal genitalia, and differentiation of external genitalia occurs between weeks 8 and 12.
</P>
				</div> <!-- div for transcription factors -->



		</div> <!-- div for the end of gene expression -->

		<div class="sub-topic">
		<h2> Mutations </h2>
		</div>

		<div class="content">

		<h3> &#10132; Genetic Mutations </h3>
				<div class="super-sub-topic">
				<button> <a href="bio docs\EXIT TICKETS - 4.1.4. a,b,c,d - Mutations - QUESTIONS.docx" > Exit Ticket </a> </button>
				<button> <a href="https://quizizz.com/join/quiz/5ed07f8c502cd9001c676b04/start?referrer=5aaf2147e860270019745570"> Quizziz for Mutations </a> </button>

				<strong class="cogverb" id="4.1.4A"> Identify  <span class="tooltiptext">
				4.1.4A: Identify <b>(= Recognise &#10132; M/C)</b> how mutation in genes and chromosomes can result from errors in:
				<uL>
					<li> DNA replication <b>(copying of DNA)</b> (point <b> (1: only affects 1 codon)</b> and frameshift mutations<b>(2: pushes/changes all following codons.)</b> )</li>
					 <li> Cell division (non-disjunction)
					</li> <li> Damage by mutagens (physical, including UV radiation, ionising radiation
					</li> <li> And heat and chemical)</li>
				</ul>

				</span> </strong>

        <button> <a href=#mut> Link to Mutagens </a> </button>
				<p> Every time a cell replicates its DNA, an error may be made. If these errors remain, this is referred to as a mutation. Mutations in the DNA during replication can be referred to as a point or frameshift mutation.
</p> <p> Occasionally an error in the DNA sequences is missed and is passed on during transcription to the mRNA.
This is translated into an incorrect amino acid in the protein that is produced. </p>
<p> <b> Mutation: </b> small permanent change in the DNA of an organism (nucleotide sequence of the genetic material).
<br><br>Mutations that result in a change in the final transcribed protein can cause the expression of a different phenotype.
</p>

<h5> Point Mutations </h5>
<p> A <b>point mutation</b> occurs inside one DNA nucleotide triplet where there is a <b>substitution of one nucleotide for another,</b> or the order within the triplet may be changed.
While this may cause changes in the amino acid sequence, the actual <b>frame of the DNA </b>strand is <b>not altered. </b>
</p>

<h5> Frame Shift Mutation </h5>
<p> If there is an <b> addition </b>of a single base (or the <b>deletion </b>of a single base) in one of these triplets, the decoding process will work perfectly until it comes to this change.
From that point on, the reading frame of three nucleotides (triplets and then the mRNA codons) will have shifted, resulting in completely different proteins being produced. This is called a <b> frameshift mutation. </b>

<br>Frameshift mutation: the deletion or insertion of a single or non-multiple of three nucleotides into the DNA sequence resulting in a shift in the frame of the following codons.
</p>
			</div> <!-- div for genetic mutations -->

			<h3> &#10132; Chromosomal Mutations </h3>
				<div class="super-sub-topic">

				<p> A <b> Chromosome mutation </b> mutation is a change in the arrangement of blocks of genes (chromosome block mutations) or the number of chromosomes (Chromosomal Non-disjunction Mutations - often due to an error in pairing during the crossing over stage of meiosis), in a cell.
		</p>


		<h5> Chromosomal Non-Disjunction Mutations </h5>
		<p><strong class="cogverb" id="4.1.4B"> Explain<span class="tooltiptext"> 4.1.4B: explain <b> (short answer: describe in more detail. )</b> how non-disjunction leads to aneuploidy <b> (how &#10132; process: steps- non-disjunction; and consequence- aneuploidy&#10132;chromosome number)</b><b></b> </span> </strong>
		<b> non-disjunction:</b> Is the failure of homologous chromosomes or sister chromatids to separate during meiosis (cellular division).
		</p>
		<p> <img src="bio photos/ane.png" width=40% class="left"> Occasionally, during meiosis (cellular division), a pair of homologous <b>chromosomes fail to separate</b> and instead travel together in <b>one direction,</b> this is how <b>non-disjunction occurs</b>.
It results in some of the gametes having two copies of a particular chromosome, while <b>other gametes</b> do not have that chromosome.
		</p>
		<p> The presence of an abnormal number of chromosomes in a cell (aneuploidy) will occur in every cell within the resultant individual.
For example, each human cell could have:<br>
<br> 45 (2n - 1) or 47 (2n + 1) chromosomes instead of the usual 46 (2n). <br> <br>
There are <b>two main divisions</b> of chromosomal aneuploidy errors: <b>autosomal aneuploidy</b>, which involves the <b>44 autosomal (body) chromosomes</b>, and sexual aneuploidy, which involves the X and Y sex chromosomes.
</p>

	<p> non-disjunction may occur during meiosis I or meiosis II. It is caused when:
		<ul>
			<li>homologous chromosomes fail to separate in meiosis I </li>
			<li> sister chromatids fail to separate during meiosis II, </li>
		</ul>
	This results in an abnormal number of chromosomes in the games referred to as aneuploidy.

	<br> NOTE: Karyotype is the number and visual appearance of the chromosomes in the cell nuclei of an organism or species
</p>

			<strong class="cogverb" id="4.1.4C"> Use <span class="tooltiptext"> 4.1.4C: <b> (2 part question) </b> use a human karyotype <b> (diagram or table) </b> to identify <b>(recognise)</b>  ploidy changes <b> (changes in chromosome numbers) </b> and predict <b>( give an expected result )</b> a genetic disorder <b> (syndrome: e.g. Trisomy 21 aka. Downs Syndrome which is three chromosomes at the 21 pair.) </b> from given data <b>(table)</b>
				 </span> </strong>

		<h5> Trisomy </h5>
			<p> <img src="bio photos/down.png" width=40% class="left"> the presence of an extra chromosome 21 (called trisomy 21) was associated with Down syndrome in humans.
This trisomy (three chromosomes) condition is not just limited to chromosome 21.
Trisomy 18 (Edwards syndrome) and 13 (Patau syndrome) are some of the syndromes that have been identified as a result of non-disjunction.

<p> see <a href="bio docs\EXIT TICKETS - 4.1.4. a,b,c,d - Mutations - QUESTIONS.docx"> exit ticket </a> for an example of using data to predict </p>
			<strong class="cogverb" id="4.1.4D"> Describe<span class="tooltiptext"> 4.1.4D: describe how inherited mutations can alter the variations in the genotype of offspring.<b></b> </span> </strong>
		<p><b>  Answer to objective: </b> (2 marks) The parent of the offspring would have a mutation that is inheritable (in sex organs) and creates a new allele (variation of a gene).
The genotype of an offspring may then be affected as it would contain new alleles  if the mutated gene from an egg or sperm is present in the zygote at fertilisation. This will then affect the genotype of the offspring.
		</P>
<br> <BR> <BR>
				</div> <!-- div for -- chromosomal mutations-->


				<h3 id="mutagens"> &#10132; Mutagens</h3>
				<div class="super-sub-topic" id="mut">
				<h5> Effects of Mutations </h5>
				<p> Mutations in the gametes of an individual, it becomes part of the genetic framework of the cell and with subsequent cell divisions will be perpetuated through DNA replication.
If the mutation occurs in a body cell, it will not be important to the species as a whole, although it could bring about significant changes to a particular organ or tissue of the individual (e.g., tumor formation).
Mutation in the sex cells may, however, affect the entire population because the mutation may be passed on to subsequent generations through the gametes.
				</p>
				<p> Gene Mutation can be classified as:
				<ol>
					<li> <b> Hereditary </b> inherited from a parent, present throughout life and found in virtually every cell. </li>
					<li> <b> Somatic </b> (acquired) may occur at some time in a person’s life in non-reproductive cells and are not passed on to the next generation. </li>
				</p>

				<p> <b> Example: </b> A mutagen is an agent, chemical or high energy radiation that changes genetic material by directly interacting with DNA.  It is therefore the agent of mutation
					</p>


				<h5> Ionizing and UV radiation </h5>
				<p> Ionising and UV radiation can <B>change the DNA structure</B>, (e.g., bends the DNA, causing the DNA to be <B>read incorrectly</B> and leading to mutation).
It can also break the DNA molecule potentially causing a mutation (e.g., cancer, if DNA repair doesn’t occur). This is more likely to occur in somatic cells that <B>cannot be transferred in the gametes.</B>
</p>

				<h5> Chemical Mutagens </h5>
				<p> There are a range of chemicals that are identified as mutagens. That is, they are <B>chemicals</B> that can <B>alter</B> the <B>structure</B> of heritable <B>DNA</B> or the process of replication.
These mutagens can <B>alter</B> the effective <B>bonding or replication</B> of genetic material leading to mutation within the genetic strand that may <B>affect</B> the <B>synthesis</B> of corresponding <B>proteins or RNA.</B>
</p>

				</div> <!-- div of mutagens -->


		</div> <!-- div for the end of mutations -->

		<div class="sub-topic">
		<h2> Inheritance </h2>
		</div>

		<div class="content">

		<h3> &#10132; Simple Inheritance</h3>
				<div class="super-sub-topic">
				<strong class="cogverb" id="4.1.5A"> Predict <span class="tooltiptext"> 4.1.5A:predict frequencies of genotypes and phenotypes using data from probability models (including frequency histograms and Punnett squares) and by taking into consideration patterns of inheritance for the following types of alleles: autosomal dominant.
<b></b> </span> </strong>

			<dl>
				<dt> Gene </dt>
					<dd> a section of DNA that codes for a particular trait or characteristic.</dd>
				<dt> Allele </dt>
					<dd> one of two or more forms of a gene located in the same position on homologous chromosomes</dd>
				<dt> Genotype </dt>
					<dd> Genotype refers to the combination of alleles that an organism has.
or in this case: A description of the pair of alleles present for a trait </dd>
				<dt> Phenotype </dt>
					<dd> The physical expression of the alleles within
a specific genotype.
</dd>
				<dt> Homozygous </dt>
				<dd> describes a condition in which both alleles for the gene are the same, e.g., HH.
</dd>
				<dt> Heterozygous </dt>
				<dd> each allele for the characteristic exhibiting a different expression.
</dd>
				<dt> Wild Type </dt>
				<dd>the most common phenotype for a feature in a population </dd>
				<dt> Pure-bred </dt>
				<dd> an organism that is homozygous for a trait and thus carries the same phenotype as the parents
</dd>
				<dt> Hybrid (genetic) </dt>
				<dd> an organism that is homozygous for a trait and thus carries the same phenotype as the parents
</dd>
				<dt> Dominant allele (autosomal dominant)
</dt>
				<dd> an allele for a gene that overrides the effects of the recessive allele in the heterozygous condition; can be autosomal or sex-linked
</dd>
				<dt> Recessive allele (autosomal recessive)
 </dt>
				<dd> an allele found on an autosomal chromosome that can be masked by its dominant form in the heterozygous condition; can be autosomal or sex-linked genotype
</dd>

</dl>
			<h4> Punnett Square </h4>
			Steps involved in the process:
				<ol>
					<li> Determine the possible genotype of each parental gamete.
					</li><li> Draw a Punnett square containing the parental gametes.
					</li><li> Determine the possible genotype of the offspring.
					</li><li>  Determine the possible phenotype of the offspring.
					</li><li>  Determine the probability of each phenotype. This can either as a ratio (eq. 3:1), probability (3/4) or as a percent (75% to 25%)
					</li>
				</ol>

					<p> The first generation of offspring is called the F1 generation. If two individuals of this F1 generation are bred with each other, the offspring are called the F2 generation.
					</p>
					<p> Dihybrid crosses: these genes may be on the same chromosome (linked) or on different chromosomes (unlinked). Unlinked chromosomes will sort independently, resulting in random combinations of the alleles for the two (or more) genes</p>

						<button onclick="myFuncton()" id="yeet">Example of Building a Punnett Square</button>

					<div id="myDV">
					<img src="bio photos/11.1A.png">
					<img src="bio photos/11.1B1.png">
					<img src="bio photos/11.1B2.png">
					<img src="bio photos/11.1B3.png">

					</div>
					<script>
					function myFuncton() {
  var x = document.getElementById("myDV");
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}		</script>


			<h5> Frequency Histograms </h5>
				<button onclick="myFuncton()" id="yet">Example of Building a Punnett Square</button>

					<div id="myDIV">
					<img src="bio photos/11.1D2.png">
					<img src="bio photos/11.1D1.png">



					</div>
					<script>
					function myFuncton() {
  var x = document.getElementById("myDIV");
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}		</script>

<h5> Pedigree Charts </h5>
<img src="bio photos/ped.png" width=70%>

				</div> <!-- div for simple inheritance -->

				<h3> &#10132; Non-Mendelian Inheritance</h3>
				<div class="super-sub-topic">
				<strong class="cogverb" id="4.1.5B"> Define and Predict <span class="tooltiptext"> 4.1.5B: define polygenic inheritance and predict frequencies of genotypes and phenotypes for using three of the possible alleles.
 </span> </strong>

				<dl>
					<dt>
						Codominance </dt>
							<dd> genetic inheritance of
								two or more traits of
								a characteristic, each
								of which is expressed
								in the phenotype equally
							</dd>
						<img src="bio photos/roan.png" width=60%>
					<h5> Incomplete dominance (intermediate or partial dominance) </h5>
					<p> a pattern of inheritance in which neither allele for a characteristic completely masks the effects of the other; results in a blending of traits for the characteristic.
</p><img src="bio photos/white.png" width=60%>

<h5> Multiple Alleles </h5>
	<p> <b> Multiple or poly alleles </b>: the inheritance of a characteristic governed by more than two allelic forms, e.g., blood groups.
</p>
<img src="bio photos/poly.png" width=50%>

<h5> Polygenic Inheritance: Continuous Variation
 </h5> <img src="bio photos/AaBb.png" width=65% class="left"><p> When a population is observed, a gradation of expression for many characteristics can be seen. Many people are not either tall or short; many mice are not either fat or thin. There are many variations between the two extremes. These situations are  explained by polygenic (or multiple gene) inheritance, where two or more pairs of alleles control a single trait. Each allele pair works according to the laws of dominance and segregation. The organism’s phenotype depends on the combined effect.
</p>
 <dl>
	<dt> Polygenic </dt>
		<dd>when one characteristic is controlled by two or more genes
 </dd>
	<dt> Continuous variation </dt>
		<dd> a variety of phenotypes as a result of more than
one gene contributing
to a characteristic.
</dd>
<dt> Gene Complexes </dt>
	<dd> two or more genes interacting to determine the phenotypic expression of a characteristic
</dd>
<dt> epistasis</dt><dd>
(= ‘standing upon’) where the phenotypic expression of one gene is masked by another gene</dd>


</dl>


	</div> <!-- div for non-mendilian inheritance -->


<h3 id="sex-linked-inheritance"> &#10132; Sex-linked Inheritance</h3>
				<div class="super-sub-topic">
				<button> <a href="bio docs\POWERPOINT - 4.1.5. a - Sex-Linked Inheritance(2).pptx"> PowerPoint for sex-linked inheritance </a></button>
				<button> <a href="https://quizizz.com/admin/quiz/5ed8c23e9979e0001e3a310a"> Quizziz for inheritance </a></button>

				<p> <b> Sex-linked genes </b> <br> Genes found on the X chromosome are termed sex-linked genes. An equivalent allele is not carried on the Y chromosome. In male offspring (XY), the X chromosome must come from the mother and the Y chromosome must come from the father. If the X chromosome in a male has an allele for a recessive trait, the trait will appear in the phenotype automatically. In contrast, females have two X chromosomes and therefore need two copies of the recessive trait allele before it is expressed in the phenotype. A heterozygous female for a sex-linked recessive gene is called a carrier .
</p> <button> <a href="bio docs\COGNATIVE VERB SCAFFOLD - Predict - (4.1.5 a) - ANSWERS A4.docx"> Practice Question with answers </a> </button>

				<p> Examples of sex-linked inheritance </p >
				<dl>
					<dt> Hemophilia </dt>
						<dd> a group of disorders in which the blood does not clot normally </dd>
					<dt> Holandric genes </dt>
					<dd> Genes found on a vertebrate Y</dd>
					</dl>

					<div class="example">
						<img src="bio photos/sa1.png" class="left" width=60%><b> SO MOCKEXAM 2020 SA 1 (4.1.5 a) QUESTION 29 (4 marks) </b>
The inheritance of Fragile X syndrome is X-linked dominant. The pedigree below shows the occurrence of the syndrome in one family with individual IV-4 being heterozygous for Fragile X syndrome. Affected individuals are shaded.

<br> <br> Predict the probability of an offspring being male with the disorder if individual IV-4 mates with an individual without the syndrome. Show your reasoning.
<br>
<p> Put your answer in here </p>
</div>


</div> <!--d iv for sex-linked inheritance -->




				</div> <!-- div for inheritance -->


<div class="sub-topic">
		<h2> Biotechnology </h2>
		</div>

		<div class="content">

		<h3> &#10132; Recombinant DNA </h3>
				<div class="super-sub-topic">


				<h4> Recombinant DNA tech </h4>
				<p> Recombinant DNA refers to the creation of new combinations of DNA segments that are not found together in nature.
The isolation and manipulation of genes allows for more precise genetic analysis as well as practical applications in medicine, agriculture, and industry.
</p>
<h5> Plasmids </h5>
<P> Plasmids are commonly used as cloning vectors (carriers). Bacterial plasmids are small, circular DNA molecules that are separate from the rest of the chromosome. They replicate independently of the bacterial chromosome.</p>
		 	<strong class="cogverb" id="4.1.6A"> Describe <span class="tooltiptext"> 4.1.6A: Describe the process of making recombinant DNA
				<ul>
					<li> Isolation of DNA, cutting of DNA (restrictive enzymes) </li>
					<li> Insertion of DNA fragment (plasmid vector) </li>
					<Li> Joining of DNA (DNA ligase) </li>
					<li> Amplification of recombinant DNA (bacterial transformation) </li>
				</ul>
<b></b> </span> </strong> 		<h4>I CREIJ LCP BTAR </h4>
				<ol>
					<li> <B> ISOLATION OF DNA </B> isolation of donor (foreign) DNA. Isolation of plasmid vector DNA. </li>
					<li> <b> CUTTING WITH RESTRICTIVE ENZYMES (ENDONUCLEASE) </B> Restriction enzymes (RE) are endonucleases that will recognise specific nucleotide sequences in the DNA and break the DNA chain at those points.
Most cut at specific palindromic sites in the DNA (sequence that is the same on both antiparallel DNA strands).
These cuts can be a staggered which generate “sticky or overhanging ends” or a blunt which generate flush ends.
</li>
<li> & 4. <b> INSERTION AND JOINING (USING THE ENZYME LIGASE) OF DNA FRAGMENT INTO PLASMID CLONING VECTOR: </B> DNAs, they must be joined together. If both have been cut with the same RE, the ends will match up because they are sticky.
DNA ligase creates a phosphodiester bond between two DNA phosphate sugar backbones.</li>
					</ol> <ol start="5"> <li start="5"> <B> BACTERIAL TRANSFORMATION AND AMPLIFICATION OF RECOMBINANT DNA </B> To recover large amounts of the recombinant DNA molecule, it must be amplified. This is accomplished by transforming the recombinant DNA into a bacterial host strain.
 When the cell divides, the replicated recombinant molecules go to both daughter cells which themselves will divide later. Thus, the DNA is amplified.
</LI>
				</div> <!-- div for recombinant DNA -->

				<h3> &#10132; PCR and Gel Electrophoresis</h3>
				<div class="super-sub-topic">
					<h4> Polymerase Chain Reaction </h4>
							<p> <b>PCR </b> = technique that permits the amplification (making multiple copies) of any short sequence of DNA, allowing it to be studied in detail. Because a small fragment of DNA can be amplified rapidly, only a very small amount of DNA is required. </p>
						<p> Used to make lots of DNA: 2<sup>no. of cycles</sup> </p>
					<h5> Application </h5>

					 <strong class="cogverb" id="4.1.6C"> Explain <span class="tooltiptext"> 4.1.6C: Explain <b> (describe in more detail: how or why)</b> the purpose <b> (WHY is it useful)</b> of polymerase <b>(used in DNA replication: makes complementary strand of nucleotides in replication)</b> chain reaction (PCR) and gel electrophoresis <b>(technique for analyzing DNA i.e.. DNA profiling) </b>
				<b><dl>
					<dt> Purpose </dt>
						<dd> how it is useful </dd> </b>
				</dl>
				</span> </strong>

										<p> PCR has a variety of applications:
It is a significant tool in the detection of infectious diseases and of variations and mutations in genes. Using primers of known DNA sequences, particular types of DNA can be detected.
</p>
		<ul>
			<li> Classification of organisms – Wildlife conservation </li> <li>
				Mutation detection- Such as pre-natal testing or diagnosis </li> <li>
				Disease diagnosis – Detection of pathogens</li> <li>
				Cancer research</li> <li>
				DNA fingerprinting <ul> <li>
				Criminal Forensics – Suspect ID</li> <li>
				Genetic matching – Paternity testing</li></ul>  </li> <li>
				Drug Discovery </li> <li>
				Genetic Engineering </li> <li>
				GMO Food detection- detection in food shipments </li>
			</ul>

			<h4> Gel Electrophoresis </h4> <img src="bio photos/elec.png" class="left mini" width=39% >
				<p> <b> Gel Electrophoresis </b> = a laboratory method for separating mixtures of DNA, RNA or proteins according to molecular size.
The short segments of DNA that have been duplicated via PCR can be used to identify differences between individual organisms via gel electrophoresis. There are many small differences between the alleles on each individual.
the same allele in different individuals will be slightly different lengths. These differences can be picked up by first cutting the DNA from the two individuals with the same restriction enzyme. Each individual will have different length fragments that result from the single nucleotide repeats. The samples can then be loaded into a thin sheet of gelatinous material (called a gel).
</p>
<p> Because DNA is negatively charged, the sample will move towards the positive terminal. Small pieces of DNA can move through the tangle of gel fibers faster than the longer lengths of DNA. This means short strands of DNA travel faster and further than the long DNA strands. . The different genes can be visualised, and measured against the DNA ladder, under ultraviolet (UV) light.
</p>

		<strong class="cogverb" id="4.1.6B"> Recognise <span class="tooltiptext"> 4.1.6B: Recognise <b> (identify &#10132; multichoice question)</b> the applications <b>(use)</b> of DNA sequencing <b> (1. technique for analysing DNA) </b> to map species’<b> (The use of technique 1) </b>  genomes and DNA profiling <b> (2. technique of analysing DNA) </b> to identify unique genetic information <b> (app. of technique 2) </b>
					<b> <dl>
			<dt> Application </dt>
				<dd> How it is used </dd>
			<dt> Unique </dt>
				<dd> Special/ Specific </dd>
			</dl> </b>
			 </span> </strong>

			<h4> DNA Profiling </h4>
				<p> DNA profiling (fingerprinting) = a technique used to identify (as for forensic purposes) the characteristics of an individual’s (or organisms) DNA by extracting and identifying the base-pair pattern of their DNA.
</p>





				</div> <!--div for eoectroj-->

				<h3> &#10132; DNA Sequencing </h3>
				<div class="super-sub-topic">

				<p> DNA sequencing involves determining the precise sequence of the individual nucleotides of an individual.
The technology behind DNA sequencing has gone through a series of changes in the last 10 years to decrease both the time and cost of the procedure.
</P >

<h4> Applications of DNA sequencing </h4>
	<h5> Evolutionary Biology: </h5>
	<p>Homologous DNA sequences from different organisms can be compared for evolutionary analysis between species or populations.
<ul> <li> generate phylogenetic trees </li> <li>
identify species or find homologous species</li> </ul>
</p>

<h5> Medicine </h5>
	<p> Comparing healthy and mutated DNA sequences can diagnose different diseases revealing  changes in a gene that may cause a disease. DNA sequencing has been used in;
<ul> <li> diagnosis of diseases</li> <li>
treatment of diseases </li> <li>
epidemiology studies</li> <li>
Identify potential drug targets</li> </ul>
</p>

		<h4> Bioinformatics </h4>
			<p> enabled massive sequences of nucleotides to be compared between individuals and species. For example, following the discovery of a previously unknown gene in a mouse, a scientist will typically perform a BLAST search of the human genome to see whether humans carry a similar gene. BLAST will identify sequences in the human genome that resemble the mouse gene based on similarity of sequence.
</p>
				</div> <!--div for DNA sequencing -->

				<h3> &#10132; Appraising Biotechnical Techniques </h3>
				<div class="super-sub-topic">

				<strong class="cogverb" id="4.1.6D"> Appraise <span class="tooltiptext"> 4.1.6D: Appraise data from an outcome of a current genetic biotechnology technique to determine its success rate.
					 </span> </strong>

					 <p> <b> COG-verb check </b>  <br>
					 Definition: to evaluate the worth or significance of something <br>
					 Application to data analysis: Appraise data to determine the success rate of methodology or techniques. Similar to evaluation </p>
					 <div class="example">
					 <img src="bio photos/a7.png" width=52% class="left">
					 <b> a) What is used as a measure as success in this study? (1 mark) </b>
					 <p> Percentage activity </p>
					 <b> b) What is the key response observed in the control? (1 mark) </b>
					 <p> insert answer </p>
					 <b> c) Compare cell lines A and B to control. For which do you have evidence that an adaptive immune response has occurred. Provide evidence for your answer (2 marks) </b>
					 <p> Insert answer here </p>

					  <b> Steps to answer question </b>
							<ol>
								<li> what is the measure of success </li>
								<li> Identify and validate control </li>
								<li> Compare treatment and control </li>
								<li> Based on step 3, what conclusions can be drawn to answer question </li>
							</ol>

					</div> <!--div for example -->




				</div> <!--div for appraising biotech techniques -->


				</div> <!-- div for biotech -->



		<div class="topic">
	<h1> Topic 2: Continuity of Life on Earth </h1>
</div>
		<div class="sub-topic">
		<h2> Evolution </h2>
	</div>
		<div class="content">
		<h3> &#10132; Evolution</h3>
				<div class="super-sub-topic">

   <p> <b> Darwin's theory </b> was based on the following observations and considerable follow-up experimentations </p>
	   <ul>
			<li> Individuals within a population have a great reproductive potential </li>
			<li> The numbers of individuals in a population remain approximately constant </li>
			<li> Variation exists within all populations </li>
			<li> Over many generations new variations gradually accumulate, possibly resulting in new species </li>
	</ul>

	<p> In natural selection of the fittest, the variation already exists within the population. Environmental pressures ‘select for’ the best-suited individuals in the population. The individuals that have the phenotypes best adapted to the environment are more likely to survive and produce offspring. Individuals that have phenotypes that are poorly adapted to the environment are ‘selected against’ and often die.
	</p> <strong class="cogverb" id="4.2.1A"> Define <span class="tooltiptext"> 4.2.1A: Define the terms evolution, microevolution and macroevolution
   </span></strong>
  <dl>
	<dt> Evolution </dt>
		<dd> the change in the genetic composition of a population during successive generations, which may result in the development of a new species. </dd>

	<dt> Microevolution </dt>
		<dd> small-scale variations in gene frequency within a species or population, in which the descendent is of the same taxonomic group as the ancestor.</dd>
	<dt> Macroevolution <dt>
		<dd> the variation of allele frequencies at or above the level of species over geological time. These variations result in the divergence of taxonomic groups, in which the descendent is in a different taxonomic group to the ancestor. </dd>
	</dl>
	<p> The process of mutations, gene flow and genetic drift affect both these types of evolution but at different scales.
</p>

	</div> <!--div for some-->

  <h3> &#10132; Evolutionary Radiation and Mass extinction </h3>
        <div class="super-sub-topic">
        <strong class="cogverb" id="4.2.1B"> Determine <span class="tooltiptext"> 4.2.1B: Determine episodes of evolutionary radiation and mass extinctions from an evolutionary timescale of life on Earth (approximately 3.5 billion years)
        </span></strong>
        <button> <a href="bio docs/ANNOTATION SHEET - Evolutionary Radiation.docx"> Other data set examples </a> </button>

        <img src="bio photos/ext.png" width=50%  style="display:block">


 </div> <!--div for evolutionary radiation and extinction -->


	<h3> &#10132; Phylogenetic Relationships </h3>
				<div class="super-sub-topic">
				<strong class="cogverb" id="4.2.1C"> Interpret <span class="tooltiptext"> 4.2.1C: Interpret data (i.e., degree of DNA similarity) to reveal phylogenetic relationships with an understanding that comparative genomics involves the comparison of genomic features to provide evidence for the theory of evolution.
   </span></strong>
   <h4> Adaptive radiation and comparative genomics </h4>
   <p> <b> Adaptive radiation </b> is driven by a single lineage’s adaptation to the environment. It is a form of divergent evolution where groups with a common ancestor accumulate mutations over time, resulting in a new species. Comparing these molecular differences is called Comparative Genomics. It involves the comparison of genomic features and provides one of many forms of evidence for evolution.
</p>
<h4> Molecular Clock </h4>
<p> The number of differences in a molecule between two species is an indication of the amount of time that has passed since they shared a common ancestor. If the mutation rate of a particular gene is known, this is called a molecular clock.
</p>

<h5> Mitochondrial DNA </h5>
<p> The mitochondrial DNA (mtDNA) is inherited directly through the maternal line (from mother to offspring), with no recombination. This means any variation is a result of an accumulation of regular mutations, producing a reliable molecular clock.
</p>
<p> The degree of evolutionary divergence of different organisms can be estimated by the number of differences between their comparable base sequences. Some genes, for example, have been found to accumulate mutations at a relatively constant rate (e.g., a 1% change per million years).
</p>

				<h4> Phylogenetic Trees </h4>
				<p> phylogenetic tree. These diagrams use the similarities between organisms (homologous structures or molecular sequences) to represent the length of time that has passed since the species shared a recent common ancestor. The individual taxa group is placed on a ‘leaf’, and the ancestral line is represented by a branch (Figure7). Each common ancestor is represented by a node (the point at which two branches combine). Each group of taxa that evolve from a common ancestor is called a clade. The beginning of the tree is the ‘root’ and represents the last common ancestor. The last common ancestor is typically unidentified on a phylogenetic tree.</p>

				</div> <!--dv class for phy rela-->
        <div class="sub-topic">
        <h2> Natural Selection  </h2>
      </div>

        <h3> &#10132; Natural Selection </h3>
      				<div class="super-sub-topic">
      				<strong class="cogverb" id="4.2.2A"> Recognise <span class="tooltiptext">
                4.2.2A: recognise <b> (identify or recall &#10132; MC or short SA question) </b> natural selection <b> (mechanism/ process of evolution) </b>
                occurs when the pressures <b> (environmental factors e.g., Abiotic or Biotic; which may change over time) </b> of environmental selection confer
                <b> (to give) </b> a selective advantage <b> (advantage over other phenotypes &#10132; physical features) </b> on a specific phenotype to enhance <b> (increased) </b>
                its survival (viability) and reproduction (fecundity)
         </span></strong>

         <p>  Natural selection, occurs when the pressure of environmental selection confers a selective advantage on a specific phenotype to enhance its survival (viability) and reproduction (fecundity)</p>
           <ul>
            <li> Organisms with heritable (genetically determined) features that help them survive and reproduce in a particular environment tend to leave more offspring than their peers.
            </li> <li> If this continues over generations, the heritable features that aid survival and reproduction will become more and more common in the population.
            </li> <li> The population will not only evolve (change in its genetic makeup and inherited traits) but will evolve in such a way that it becomes adapted, or better suited, to its environment.
            </li>
          </ul>
          <strong class="cogverb" id="4.2.2B">Identify <span class="tooltiptext"> 4.2.2B: Identify that the selection of allele frequency <b> (number of individuals with a specific allele) </b>
            in a gene pool <b> (Population and all their genes) </b> can be positive <b> (increase the frequency of allele) </b> or negative <b> (decrease the frequency of the allele) </b>
                      </span></strong>

                  <p> The ultimate source of the phenotypic variability on which natural selection works is mutation.
                  Microevolution describes the change in the allelic frequency (in the genotype) in a population over a succession of generations.
                  </p><p> By acting on the phenotypes of individuals, natural selection inadvertently increases the frequency of favorable alleles (positive selection) and decreases non-beneficial alleles (negative selection) in the gene pool (genotype of the population). This change in allelic frequency results in microevolution of the population.</p>
                  <h4> Fitness = Reproductive success </h4>
                    <p> Fitness is a measure of how well organisms survive and reproduce, with emphasis on "reproduce." Officially, fitness is defined as the number of offspring that organisms with a particular genotype or phenotype leave behind, on average, as compared to others in the population.</p>

                    <div class="box">
                      <h4> Exit Ticket </h4>
                      <p> <h5> PAST EXAM 2020 SA 2
                  QUESTION 9 (3 marks) </h5> <img src="bio photos/alefr.png" class="left" width= 40%>
                  Fossil evidence seems to show that the morphology of the Queensland lung fish has remained relatively unchanged for the past 100 million years.
                  Describe the features of the theory of natural selection to explain how this may have occurred.
                  <br> <br> My answer: They have not had environmental stressors
                      </p>
                      <p> <h5> PAST EXAM 2020 MC QUESTION 18</h5>
                       The diagram shows the abundance (n) of phenotypic and genotypic variation for the colour trait of a species of butterfly.
                  If these butterflies were placed in an environment with light-coloured trees, which option from the table would be the most likely effect on the allelic frequencies?
                  <br> <br> My answer: c
                      </p>
                    </div>

                  </div>

                  <h3> &#10132; Phenotypic Selection </h3>
                        <div class="super-sub-topic">
                        <strong class="cogverb" id="4.2.2C"> Interpret <span class="tooltiptext">
                          4.2.2C: <b> (PART 1) </b> Interpret <b> (1. recognise trends and 2. draw conclusions &#10132; say what the trend means) </b> data <b> (graph) </b> and <B>(Part 2) </b> describe the three main types of phenotypic selection: stabilising, directional and disruptive.

                            <br> <br> <B> Describe the Three forms of phenotypic selection in words </b>
                              <br> Probably going to give a polygenic trait </br>
                            </b>

                            </select>
                   </span></strong>

                  <h4> 1. Stabilizing Selection </h4>
                  <p> natural selection for a particular variant (mean) of a characteristic that maintains consistency of a population; occurs in times of environmental stability.
                  Stabilizing selection keeps the same mean but tends to narrow the curve.</p>
                  <img src="bio photos/stabs.png" width=55%>

                  <h4> 2. Directional Selection </h4>
                  <p> natural selection that favours one phenotypic extreme causing the allele frequency to shift over time in the direction of that phenotype. i.e. one extreme phenotype is more fit than all the other phenotypes.
                    Directional selection shifts the curve towards the favorable phenotype.</p>
                    <img src="bio photos/dirs.png" width=55%>

                  <h4> 3. Disruptive Selection </h4>
                  <p> natural selection favouring individuals with extreme phenotypes. Both extreme phenotypes are more fit than those in the middle. Diversifying selection makes multiple peaks in the curve. </p>
                  <img src="bio photos/dis.png" width=30%>


                    <div class="box">
                      <h4> Exit Ticket </h4>
                      <p> <h5> PAST EXAM 2020 SA 2 Question 10 (3 marks) </h5 >
                        <img src="bio photos/4.2.2C.png" class="left" width=30%>
                        Researchers measured the adult beak lengths of an entire population of a species of bird and plotted their results on the graph.
                      After many generations, the lengths of the adult beaks were again measured. By comparing this new data to the original data, the researchers concluded that the average length of beaks had increased as a result of directional selection.
                      </p>
                      <p> a) Describe directional selection. [2 marks]
                        <br> <div class="answer">
                          Natural selection that favours one phenotypic extreme causing allele frequency to shift over time in the direction of that phenotype.
                        </div>
                      </p>

                      <p> b) On the graph provided, sketch a representation of directional selection for the beak length scenario. [1 mark]
                        <br>
                        <div class="answer">

                        </div >
                      </p>

                    </div>


                  </div> <!--div for phenotypic selection -->

                  <h3> &#10132; Microevolutionary Processes</h3>
                  <div class="super-sub-topic">
                  <strong class="cogverb" id="4.2.2D">Explain <span class="tooltiptext"> 4.2.2D:
                    explain microevolutionary <b> (change in allelic frequency &#10132; change in the number of individuals withing a population with a specific allele or group of alleles) </b>
                    change through the main processes of mutation <b> (probably not mutation in exam) </b>, gene flow and genetic drift.
                  </span></strong>




                  <h5> Microevolution Through mutations</h5>
                  <p > <b style="color:F49F0A;"> Defining microevolution: </b> microevolution describes the <b> change in the allelic frequency </b>in a population over a succession of generations.
                  </p>
                   <p class="bold"> New alleles must arise through <b>random mutation</b> over many generations, <b>resulting in new phenotypes</b>.
                  The principal role of new mutations is in replenishing the store of variability in the gene pool.
                  Mutations (and their alleles) that <b>enhance survival </b>as a result of changes in environmental pressures over time, will<b> give selective advantage.</b>
                  Natural selection will <b>increase</b> their <b>frequency</b> in the population.
                  Consequently, there will be a change in the frequency of particular advantaged alleles within a species and Microevolution will have occurred. </p>



                  <h5> Microevolution through gene flow </h5>
                    <p class="bold"> <b style="color:F49F0A;"> Defining gene flow </b> (also known as gene migration or allele flow) the transfer of genetic variation from one population to another. If the rate of gene flow is high enough, then two populations are considered to have equivalent allele frequencies and therefore effectively be a single population.
                  </P>
                  <p class="bold">
                  Gene flow describes of <b> movement of genes  </b>between populations.
                  If the  <b>mixing of genes from previously isolated populations  </b>that have diverged occurs, this can rapidly change gene frequencies in the newly merged population(s).
                  Consequently, there will be a <b>change in the frequency of particular advantaged alleles </b> within a species and Microevolution will have occurred.
                  An example of this would be 'Migration'.
                  </p>



                  <h5> Microevolution through Genetic Drift</h5>
                  <p class="bold"> <b style="color:F49F0A;"> Defining genetic drift: </b> the variation in the relative frequency of different genotypes in a small population, owing to the chance <b> disappearance </b> of particular <b> genes </b> as individuals <b> die or do not reproduce.</b>
                  </p>
                  <p class="bold"> Genetic drift describes <b>  changes in the gene pool </b> , and thus the relative frequencies of different alleles, in a population due to chance.
                This process is generally common in <b> small </b> and or <b> isolated populations</b>  where <b> gene frequencies</b>  can <b> change</b> rapidly due to <b> random events</b>  that are more likely to significantly change gene frequencies in small population such as <b>  environmental disasters.</b>
                An example of this would be 'Founder Effect‘ or ‘Bottlenecks’.
                </p>

                  </div> <!-- div for microevolutionary changes -->




                  <div class="sub-topic">
                  <h2> Speciation and Macroevolution </h2>
                  </div>
                  <h3> &#10132; Diversification and macroevolution</h3>
                      <div class="super-sub-topic">
                        <strong class="cogverb" id="4.2.3A">Recall <span class="tooltiptext"> 4.2.3A: Recall <b>(MC or low mark SA)</b> that speciation and macroevolutionary changes result from an accumulation of microevolutionary changes over time
                          <span></strong>
                            <p> Speciation and macroevolutionary changes result from an accumulation of these microevolutionary changes in phenotypes over time, resulting in new species being formed and over greater periods of time entirely new taxonomic groups.
                  </p>
                  <strong class="cogverb" id="4.2.3B"> Identify<span class="tooltiptext"> 4.2.3B: Identify that diversification between species can follow one of four patterns:
                  divergent, convergent, parallel and coevolution

                    <span></strong>
                  <p> There are four key patterns that evolution can take for the diversification in phylogeny of the taxonomic group:
                    <ul> <li> divergent evolution</li> <li>  parallel evolution</li> <li> convergent evolution </li> <li>coevolution</li> </ul>

                  </p>

                  <h5> 1. Divergent Evolution </h5>
                  <p>Divergent evolution is the diversification of a shared ancestral organism into two or more species that are different in form from their common ancestor, experience different environmental pressures, have different niches and are geographically or ecologically isolated.
                  When divergent evolution involves the formation of a large number of species to occupy different niches this is called an adaptive radiation.
                  </p>

                  <h5>2. Parallel Evolution </h5>
                  <p> Parallel evolution occurs when two species that once shared a <b>common ancestor</b> are exposed to similar environmental pressures and as a result, similar form or phenotypic features are selected for, despite their geographical isolation.
                  </p>
                  <h5>3. Convergent Evolution </h5>
                  <p> <img src="bio photos/conv.png" class="left"> Convergent evolution is the independent development of similarities form or phenotypic features between species from different evolutionary branches (different ancestors), as a result of them having similar niches and selection pressures, they may be geographically, ecologically or temporally (time) isolated.
                  </p>

                  <h5>4. Coevolution </h5>
                  <p> Coevolution occurs when two or more species reciprocally affect each other’s evolution.
                  Coevolution results in simultaneous phenotypic adjustments between community members. When two species are very reliant on each other, each species exerts a strong selective force on the other. These interactions are mainly between predator or parasite and prey.
                  </p>

                  <img src="bio photos/pat.png" width=60%>
                  <img src="bio photos/pat2.png" width=60%>

                  </div> <!--div for diversification -->


                  <h3>&#10132; Modes of Speciation  </h3>
                  <div class="super-sub-topic">
                            <strong class="cogverb" id="4.2.3C"> Describe <span class="tooltiptext"> 4.2.3C: describe the modes of speciation: allopatric, sympatric, parapatric
                            </span></strong>
                  <p> Speciation in general is defined as the evolution of strong, genetically based ‘isolating barriers’ between populations that prevent them from exchanging genes, for example, mate discrimination, habitat differences and hybrid sterility.
                  </p>
                  <h5> Allopatric Speciation </h5>
                  <p> Allopatric speciation is described as;
                  when two subpopulations of a species become geographically and genetically, completely isolated.
                  Each is subjected to different natural selection pressures, and they establish reproductive isolating mechanisms, forming truly separate species.
                  Barriers might include; mountains, rivers or seas.
                  </P>

                  <h5>Sympatric speciation </h5>
                  <p> Sympatric speciation is described as;

                  the splitting of a single ancestral species into two or more reproductively isolated groups without geographical isolation of those groups but with little or no genetic exchange.
                  Reproductive isolation might be a consequence of subtle differences in; behaviour, structure, and physiology.
                  </p>

                  <h5> Parapatric speciation </h5>
                  <p> Parapatric speciation is described as;
                  when two subpopulations are only partially separated geographically, with one smaller population usually at the periphery (side) of a larger group. They evolve reproductive isolating mechanisms while still exchanging some genes along a shared border zone. So, the population is continuous but has non-random mating
                  </p>

                    <img src="bio photos/compar.png" width=60% >
                              </div> <!--modes of speciation -->

                                <h3> &#10132; Isolation Mechanisms </h3>
                                <div class="super-sub-topic">
                                <strong class="cogverb" id="4.2.3D"> Understand <span class="tooltiptext"> 4.2.3D:
                                  understand that the different mechanisms of isolation — geographic (including environmental disasters, habitat fragmentation),
                                  reproductive, spatial, and temporal — influence gene flow
                                </span></strong>

                                    <h5> Reproductive </h5>
                                        <p> Reproductive Isolating mechanisms (RIMS) prevent successful breeding between different species. They are barriers to gene flow.
                                        </p>
                                        <p> An alternative mechanism for isolation of a species is Geographical barriers, these prevent species interbreeding but are not considered to be RIMs because they are not operating through the organisms themselves.
                                        </p>
                                      <h5> Geographical</h5>
                                      <p> Geographical isolation describes the isolation of a species population (gene pool) by some kind of Geographical or physical barrier that prevent interbreeding with related species or populations.
                                          Geographical barriers include mountains, rivers, and oceans. </p>

                                      <h6> Habitat fragmentation</h6>
                                      <p> habitat fragmentation is a process by which large and contiguous habitats get divided into smaller, isolated patches of habitats.
                                      </p>
                                      <p> <b> Causes of this may include </b>
                                        <ul> <li> Natural fragmentation cause: geographical processes that slowly alter the layout of the physical environment (suspected of being one of the major causes of speciation), such as rising sea levels.
                                  </li> <li> Human cause: human activity such as land clearing, infrastructure development and urbanisation, which can alter, destroy or interrupt habitats . This may result in the eradication of species or isolation of small populations or the interruption of gene flow between once connected sections of habitat.
                                  </li> </ul>

                                  <h6> Environmental Disasters </h6>
                                  <p> Environmental disasters, can damage, change or fragment habitats resulting in isolation of populations. Examples are floods and bushfires.</p>

                                  <h5> Spatial Isolating Mechanism </h5>
                                  <dl>
                                    <dt> Spatial (ecological or habitat)</dt>
                                    <dd> Ecological or habitat isolation may describe a barrier between two species as a result of them occupying different habitats within the same geographical
                                    </dd>
                                    <dt> temporal</dt>
                                    <dd><ul> <li> Temporal isolation means isolation in time. Species may have different activity patterns; they may be nocturnal or diurnal, or breed at different seasons.
                                </li> <li> In this hypothetical example, the two species of butterfly will never mate because they are sexually active at different times of the year.
                                  </li> </ul ></dd>
                                  <dt> Behavioral </dt>
                                  <dd> Species may have specific calls, rituals, postures etc. that enable them to recognize potential mates (many bird species have elaborate behaviors).
                                </dd>
                                  <dt> Structural </dt>
                                  <dd> For successful mating,
species must have compatible copulatory apparatuses, appearance, and chemical make-up (odor, chemical attractants</dd>
                                  <dt> Gamete Mortality</dt>
                                  <dd> If sperm and egg fail to unite, fertilization will be unsuccessful.
                                </dd>
                                </dl>
                              </div>


                <h3>&#10132; Population Bottlenecks </h3>
                <div class="super-sub-topic">

                      <h4> Reduced Genetic Diversity</h4> <p>
                        Populations may sometimes be reduced to low numbers by predation, disease, periods of climatic change or through geographic isolation from the greater population (reduced gen flow or founder affect). although population size might return to previous levels, genetic diversity might not. Such population may suffer from the effects of reduced genetic diversity.
                                          </p>
                                            <p> A population bottleneck is defined as; an event that drastically reduces the size of a population.
                    The bottleneck may be caused by various events, such as an environmental disaster, the hunting of a species to the point of extinction, or habitat destruction that results in the deaths of organisms.
                    </p>

                    <h4> Importance of Genetic Diversity</h4>
                    <p> The reason that population bottlenecks (reduced population size), and consequently genetic bottlenecking (reduction in genetic diversity of a population), is significant, is that genetic diversity increases individual fitness (viability and fecundity) and provides the pool of potential alleles for populations to adapt to changing environments. With more variation, it is more likely that some individuals in a population will possess variations of alleles that are suited for the environment. Consequently, increased diversity increases the resilience of the species to change.</p>
                    <strong class="cogverb" id="4.2.3E"> Explain <span class="tooltiptext"> 4.2.3E: explain <b> (may be given with a story or context behind it) </b> how <b> (because explain how show the steps)</b> populations with reduced genetic diversity (i.e. those affected by population bottlenecks <b> (reduced population size which causes reduced genetic diversity) </b>) face an increased risk of extinction
                      <span></strong>
                    <h4> Reduced diversity leading to risk of extinction</h4>
                    <P><B>How can reduced diversity during a population bottleneck led to increased risk of extinction (explained):</B> <BR>
                    <br> Population bottlenecks result from a large-scale reduction in population size in a species that might occur as a result of natural catastrophes, disease or habitat loss.
                    This may result in a simultaneous loss of genetic diversity across the surviving population (genetic bottleneck).
                  </p>
                  <p><b> ANSWER </b> <br>
                      <b> Reduced genetic diversity </b> results in increased levels of inbreeding leading to <b>lower levels of individual fitness </b>(viability and fecundity) and therefore lower rates of reproduction. This is a result of increased rates of negative alleles.


                    The <b>decrease in genetic diversity </b>in terms of the variety of inheritable alleles across the population results in a <b>decrease</b> in the population’s <b>ability to adapt </b> to <b>environmental changes.</b>


                     populations that suffer from reduced genetic diversity have an increased likelihood of experiencing extinction due to catastrophic environmental events, disease or large scale ecological or climatic shifts.
                  </p>
                    </div> <!--div for population bottlenecks -->




                    <h3>&#10132; Determining Speciation </h3>
                    <div class="super-sub-topic">

                          <h4> Gene Flow</h4>
                           <p>Gene flow — also called migration — is any movement of individuals, and/or the genetic material they carry, from one population to another. If gene versions are carried to a population where those gene versions previously did not exist, gene flow can be a very important source of genetic variation. If gene flow occurs between sub populations, gene frequencies are not changed for the species as a whole, but they change locally whenever different populations have different allele frequencies
                           </p>

                           <h5> consequences of gene flow changes </h5>
                           <p> If the rate of gene flow is high enough, then two populations are considered to have equivalent allele frequencies and therefore effectively be a single population.
                             When gene flow stops, the chances of there being evolution due to genetic drift or environmental pressure is increased. The effect of genetic drift is strongest in small populations creating divergence and possible speciation.
                            Stop in gene flow can result in an increase in inbreeding, within a population
                            </p>

                      <strong class="cogverb" id="4.2.3F"> Interpret<span class="tooltiptext"> 4.2.3F: interpret <b> (1.indentify trends patterns and relationships and state them  2. Draw a conclusion &#10132; i.e. what does this mean) </b>gene flow and allele frequency data from different populations in order to determine speciation.
                        <span></strong>

                          <ul>
                            <li> Direction of gene flow ---> </li>
                            <li> Each circle represents a separate subpopulation of a given species. </li>
                            <li> Area of the circle represents population size </li>
                          </ul>


</div> <!--div for speciation and macroevolution-->

</div> <!-- content: started from evolution at the top -->  <!---div for division two -->
			<!--Separator for the end -->
      <div>
        <H1 id="definitions"> Definitions </h1>
          <div class="super-sub-topic">
            <h2> Unit 3 </h2>
            <dl>
              <a href=#3.1.2C><dt>Clade
              </dt></a> <dd>a group of organisms that consists of a common ancestor and all
its lineal descendants
</dd> <a href=#3.2.1E><dt> Ecological Species
              </dt></a> <dd>the role and space that an organism fills in an ecosystem,
including all its interactions with the biotic and abiotic factors of its
environment
</dd> <a href=#3.2.1H><dt> Keystone species
              </dt></a> <dd>a plant or animal that plays a unique and crucial role in the way an
ecosystem functions
</dd> <a href=#3.2.2A><dt> Carrying Capacity
              </dt></a> <dd> in Biology, the size of the population that can be supported
indefinitely on the available resources and services of that
ecosystem
</dd></dl>
<h2> Unit 4 </h2>
<dl>
             <a href=#4.1.3A><dt> Gene
              </dt></a> <dd>region/s of DNA that are made up of nucleotides; the molecular
unit of heredity
</dd> <a href=#4.1.3A><dt> Genome
              </dt></a> <dd>region/s of DNA that are made up of nucleotides; the molecular
unit of heredity
</dd> <a href=#4.1.5B><dt> Polygenic inheritance
            </dt></a> <dd> when one characteristic is controlled by two or more genes
            </dd> <a href=#4.2.1A><dt> Evolution
            </dt></a> <dd> change in the genetic composition of a population during
successive generations, which may result in the development of
new species
            </dd> <a href=#4.2.1A><dt> Microevolution
            </dt></a> <dd>small-scale variation of allele frequencies within a species or
population, in which the descendant is of the same taxonomic
group as the ancestor
</dd> <a href=#4.2.1A><dt> Macroevolution
</dt></a><dd>the variation of allele frequencies at or above the level of species
over geological time, resulting in the divergence of taxonomic
groups, in which the descendant is in a different taxonomic group
to the ancestor

            </dd></dl>

		<br>
		<br>
    <div class="super-sub-topic">
    </div>
<h3 id="letterstrings"> Letter Strings </h3>
<div class="content">
  <dl>
    <dt>      P COFGs
    </dt><dd> Classification structure: Phyla, class, order, families, genus and species
    </dd><dt> PSMT BPA
    </dt><dd> Sampling requirements: <b>p</b>urpose, <b>s</b>ite selection, sampling <b>m</b>ethod and <b>t</b>echnique, <b>b</b>ias minimisation, <b>p</b>resentation, <b>a</b>nalysis
    </dd><dt> I CREIJ LCP BTAR
    </dt><dd> Recombinant DNA: Isolation and cutting (restrictive enzyme, endonuclease), insertion and joining (ligase) to plasmid cloning vector, bacterial transformation and amplification of recombinant DNA
    </dd><dt> GF IPM FCA
    </dt><dd> Microevolution via gene flow: Isolated populations mix causing  frequency change (allele) of advanced alleles.
    </dd><dt> GD SIP RE DG &#10132; gee dee sip redig
    </dt><dd> Microevolution via genetic drift: small, isolated population &#10132; random event &#10132; disappearance of genes
    </dd><dt> PMAT
    </dt><dd> Prophase, metaphase, anaphase, telophase
    </dd><dt> CTIP &#10132; see tip
    </dt><dd> Classification of ecosystems allows communication on global scale, Tailor management, Identify and protect vulnerable ecosystems
</div>
		</div> <!--div for everything for the end  -->

		</div>
</div></div>


    </div>

    <div id="div-placeholder">

    </div>

        <script>
        $(function(){
          $("#div-placeholder").load("../footer.html");
        });
        </script>

	</body>
</html>
